app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -690,11 +692,20 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                 }
             )
         }
-        CustomizeHome.jumpBackIn.set(settings.showRecentTabsFeature)
-        CustomizeHome.recentlySaved.set(settings.showRecentBookmarksFeature)
-        CustomizeHome.mostVisitedSites.set(settings.showTopFrecentSites)
-        CustomizeHome.recentlyVisited.set(settings.historyMetadataUIFeature)
-        CustomizeHome.pocket.set(settings.pocketRecommendations)
+        reportHomeScreenMetrics(settings)
+    }
+
+    @VisibleForTesting
+    internal fun reportHomeScreenMetrics(settings: Settings) {
+        components.analytics.experiments.register(object : NimbusInterface.Observer {
+            override fun onUpdatesApplied(updated: List<EnrolledExperiment>) {
+                CustomizeHome.jumpBackIn.set(settings.showRecentTabsFeature)
+                CustomizeHome.recentlySaved.set(settings.showRecentBookmarksFeature)
+                CustomizeHome.mostVisitedSites.set(settings.showTopFrecentSites)
+                CustomizeHome.recentlyVisited.set(settings.historyMetadataUIFeature)
+                CustomizeHome.pocket.set(settings.showPocketRecommendationsFeature)
+            }
+        })
```

Comment:
I need this for testing purpose, as the call to  `experiments.register` was causing some native code to be executed, causing the test to failed.

See [Task Cluster logs ](https://pipelines.actions.githubusercontent.com/fpQhDo7gFGAi2U5nnAKDDfkyxMq3ujEuynV0EYZitkpSpYyION/_apis/pipelines/1/runs/36381/signedlogcontent/6?urlExpires=2021-10-01T03%3A12%3A47.1750420Z&urlSigningMethod=HMACV1&urlSignature=jAqJDHcnEq1QiVjBtofwWoz%2BhCceD%2Bz9YqsUE%2BGzQ84%3D)

```

2021-10-01T02:51:31.5638661Z SUITE: org.mozilla.fenix.FenixApplicationTest
2021-10-01T02:51:31.6636396Z   TEST: WHEN setStartupMetrics is called THEN sets some base metrics
2021-10-01T02:51:31.6638033Z     E/glean/Dispatchers: Exceeded maximum queue size, discarding task
2021-10-01T02:51:32.1640784Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.1642404Z     D/SQLiteCursor: received count(*) from native_fill_window: 1
2021-10-01T02:51:32.1643596Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.1644756Z     D/SQLiteCursor: received count(*) from native_fill_window: 1
2021-10-01T02:51:32.2636398Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.2637933Z     D/SQLiteCursor: received count(*) from native_fill_window: 1
2021-10-01T02:51:32.2639153Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.2640314Z     D/SQLiteCursor: received count(*) from native_fill_window: 0
2021-10-01T02:51:32.2641469Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.2642626Z     D/SQLiteCursor: received count(*) from native_fill_window: 0
2021-10-01T02:51:32.2643781Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.2645231Z     D/SQLiteCursor: received count(*) from native_fill_window: 0
2021-10-01T02:51:32.3636160Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.3637860Z     D/SQLiteCursor: received count(*) from native_fill_window: 0
2021-10-01T02:51:32.3639173Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.3640287Z     D/SQLiteCursor: received count(*) from native_fill_window: 0
2021-10-01T02:51:32.3641747Z     I/CursorWindowStats: Created a new Cursor. # Open Cursors=1 (# cursors opened by this proc=1)
2021-10-01T02:51:32.3642821Z     D/SQLiteCursor: received count(*) from native_fill_window: 0
2021-10-01T02:51:32.4636884Z     E/glean/MetricsPingSched: MetricsPingScheduler last stored ping time was not valid
2021-10-01T02:51:32.4639872Z     I/glean/MetricsPingSched: The 'metrics' collection is scheduled for today, 1633056692357
2021-10-01T02:51:32.4641383Z     D/glean/MetricsPingSched: Scheduling the 'metrics' ping in 4107643ms
2021-10-01T02:51:33.3636147Z     E Nimbus error: Failed to initialize Nimbus
2021-10-01T02:51:33.3638317Z     java.lang.UnsatisfiedLinkError: Unable to load library 'megazord':
2021-10-01T02:51:33.3640044Z     libmegazord.so: cannot open shared object file: No such file or directory
2021-10-01T02:51:33.3641839Z     libmegazord.so: cannot open shared object file: No such file or directory
2021-10-01T02:51:33.4798387Z     Native library (linux-x86-64/libmegazord.so) not found in resource path ([file:/home/runner/.gradle/caches/7.0.2/workerMain/gradle-worker.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/unit_test_config_directory/debugUnitTest/out/, file:/home/runner/work/fenix/fenix/app/build/intermediates/javac/debugUnitTest/classes, file:/home/runner/work/fenix/fenix/app/build/tmp/kotlin-classes/debugUnitTest/, file:/home/runner/work/fenix/fenix/app/build/intermediates/java_res/debugUnitTest/out/, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna/5.6.0/330f2244e9030119ab3030fc3fededc86713d9cc/jna-5.6.0.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/app_classes/debug/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ab4ce8d4ea778cadc4a1e5739fe6ae68/transformed/support-test-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-test/1.5.0/403d7911978eb028ddf0c864cc7b22807ecf7588/kotlinx-coroutines-test-1.5.0.jar, file:/home/runner/.gradle/caches/transforms-3/65dc2cf90a17989e538ba22b2ce1c189/transformed/support-test-libstate-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ae38b520f022b18920a7d654f0e8c0d2/transformed/junit-ktx-1.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/141918be8bd74d2b2f8e4a0e90f9b4b9/transformed/junit-1.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8ec5044bfffb9044c3271592ae760fff/transformed/work-testing-2.5.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.6.1/357f1e5ae699fdaaf02cc8c2152922d4e69dcbbb/robolectric-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-ant-tasks/2.1.3/b09be554228d66d208e5fef5266844aacf443abc/maven-ant-tasks-2.1.3.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk/1.12.0/5c86b2753ab6484e98f93419bf2189927184a575/mockk-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.mozilla.telemetry/glean-native-forUnitTests/41.1.1/bea27df1f95ac9020269427c7f2391e165f6ff4/glean-native-forUnitTests-41.1.1.jar, file:/home/runner/.gradle/caches/transforms-3/3524a701b04aa691f6f04ad22f38635f/transformed/feature-pwa-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d7f02fc2f8561480c836c902f4114a92/transformed/feature-customtabs-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/4c411c3d682790bfdca42cad443a424c/transformed/browser-toolbar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a1ae370a9b59b133e3bfbcc62f9dd3e5/transformed/feature-readerview-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/71d4820a670fb3183727c5692dd9adf4/transformed/browser-menu-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/950dd39f407cc907f61b60df596ce069/transformed/support-migration-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9fdd40e310786d6695a8685a766f6299/transformed/feature-addons-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f7dc662f2f632200debabdae9fc85fa3/transformed/feature-downloads-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/cf703b5de6670cb2173282302c32b85b/transformed/compose-awesomebar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/80edf3ccc96e856cb42cb20c27d10b77/transformed/feature-awesomebar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e6c888715d3127e21402e3265655b5fb/transformed/feature-syncedtabs-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f4a19639e6f4386d212c18c6aba740bc/transformed/concept-awesomebar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5fdf6790a24d5b5b93052124d6443993/transformed/browser-engine-gecko-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d08e729d70f3424ac5fe8a9cab5ca5b0/transformed/browser-tabstray-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7c7c52d6c6fe4b22cdcb1aaa8463f65c/transformed/feature-accounts-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e870e196d5221299ea0adae4505c92e1/transformed/feature-contextmenu-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/90f8c7c3aaf8998aa52d5dd3a03a3733/transformed/feature-intent-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ca207be4f1a919b0b45aa163cbec0acd/transformed/feature-search-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d6757ede70c7439b79b648cec602e9b0/transformed/feature-tab-collections-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c58021f21d1f0a69bdc1d35133b8f73c/transformed/feature-tabs-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/eaffbe479c41fa99a94ed27d1345fda9/transformed/concept-tabstray-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e67d804e60e9b678c89af7d30cbbb99e/transformed/feature-webnotifications-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9aab0070377b7b546a8437611877b82a/transformed/browser-icons-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/70c0a0c2456e84e208d9ae91fb84c3e3/transformed/browser-session-storage-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/21bb6129d1db0fbe5046e130dfda2ea1/transformed/browser-thumbnails-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d1b775ecbc4b8ecf7b89bb6467843c81/transformed/feature-app-links-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/89a172948b7df043e71475f83eb0bbb5/transformed/feature-media-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/46bc8b6ab8f2763b175b4fc5099c7b2e/transformed/feature-logins-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/07f2289746b8be7cb411cd5e188253f1/transformed/feature-prompts-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ae75793e4d226a6fc79ef3fb14511391/transformed/feature-privatemode-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8f08d462e4393c72d3ce55ef7d99ea89/transformed/feature-toolbar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9b31171dbf1682ced5e92f9ef65e4497/transformed/feature-session-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8517e9c5cc3b30977861667c6a2fb675/transformed/feature-findinpage-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c42b24f93f0d2d67fc4ec11d42dc70b0/transformed/feature-sitepermissions-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ee248d6c3401f6f4c8f64db47b1ee979/transformed/feature-recentlyclosed-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/83ee3792166b978dcf723f91dac82ce5/transformed/feature-webcompat-reporter-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d3a0de4ba68a5299a9a17fa737f07cd7/transformed/support-webextensions-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/12701a95bddb43b9a22e2e5b7e38b267/transformed/service-nimbus-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3afc47fd0f7461f2d05aec0402f4e786/transformed/support-locale-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/276c58c73a85eca3005f24ae345c5488/transformed/browser-state-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/bdf5c85dcef6a2eb2afdd8cb89bdf07f/transformed/feature-webauthn-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/45597e18a40acd4620116682c9272e61/transformed/feature-webcompat-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/2708d125a2c7a7f9767d1ac444cccf3f/transformed/concept-engine-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b05d2f7bd3cd494e1d56120ce192da45/transformed/ui-tabcounter-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f8d329ef84c1cd75cc9913ce6e76ada8/transformed/browser-menu2-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/dbc299f7a1a717dcb0acb9b6d3d309bf/transformed/concept-menu-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d7db98ac290c914752d867a6279bec51/transformed/feature-accounts-push-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6aafcb230cffd09d12908ae459b68837/transformed/feature-push-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0b8d24cfaa7a9ad6c5b9b4412b8ce5a7/transformed/lib-push-firebase-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/2075835291c8ddf981608d06f9ad406c/transformed/concept-push-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1795cdb03e6008c80db8807f2fcf1151/transformed/feature-top-sites-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/24c6b330f0354b1c51c38df1964c580d/transformed/browser-storage-sync-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/59296be3932680db4c37b36b14851566/transformed/service-sync-autofill-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6f21ed8c3d76fe1cc9f282608ac5e24b/transformed/service-sync-logins-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0ac4393f4ef96652c71cf7a105f98078/transformed/service-firefox-accounts-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3fb6299bf9ea75263c17e5980c9db657/transformed/concept-sync-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6cfb159886a2465ea0c98cdb92fa654b/transformed/concept-toolbar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9f31ffbad5eca6d4f179ce5558bb2ba6/transformed/feature-autofill-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/972754c27294d8316808914a95879ad2/transformed/feature-qr-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7aae232278351f364ec74b5136193fe0/transformed/feature-share-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3a696d48b84317c774933bac7165272c/transformed/service-pocket-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ffa8d3870519f2d39844395dbde14453/transformed/service-digitalassetlinks-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/816d30dfd653075f1995a33fbac3296d/transformed/support-sync-telemetry-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e3311a982617d3d68004f78d91ab90ea/transformed/service-glean-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/edc85318e0e8b83b8dcadd3d00a133a1/transformed/service-location-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f99eef156b829698ff66b097e52cf270/transformed/support-images-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/801c0f6394c02f2e9655c2cb8a090751/transformed/lib-publicsuffixlist-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/cf47cd80caa2a9593fdf672ca036ef23/transformed/lib-crash-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/79a6fa67d1e905fa967934a7819474b4/transformed/lib-state-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f421d318749a6c09439355a24764b81b/transformed/browser-errorpages-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1d89a3aee6968ddc940dbc5fd88619c8/transformed/support-ktx-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/93d963a192b227465184c3514e40dee1/transformed/support-rustlog-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6100b5e88e9ef31505d36e378238818e/transformed/ui-autocomplete-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/85be3d85b32564d90653ec046fbf5333/transformed/support-utils-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e1e9a0d41214857ea0819035b1f10ed9/transformed/lib-dataprotect-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f4c953affcc57e288dc75b5d8645c8a0/transformed/support-base-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5c84a50be54943650591fa024a3decd7/transformed/viewbinding-7.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/746683e77e802d040d13e5df144d71a7/transformed/leakcanary-android-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6e55b16e64797af0a5c9b685528ca43f/transformed/concept-base-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0207f98fddae894a47d7562fc011187e/transformed/concept-storage-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-parcelize-runtime/1.5.10/37f48be9c9be4b8c74bf5a5aa17050c18581a370/kotlin-parcelize-runtime-1.5.10.jar, file:/home/runner/.gradle/caches/transforms-3/afd83d015a267446ad132cd48bffd600/transformed/places-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b6bdedaeae68891d4b8cb3696c38ed80/transformed/tabs-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/891a4fbb6e6434cf249634a5729c27ca/transformed/push-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7853a5be97e3cbb3242b451213ec386d/transformed/autofill-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/234375d562c52a9def711ce1566c40a2/transformed/logins-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f2dae8d08e73bc75ff54a05174de27d6/transformed/syncmanager-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a5da0ec537ce1ca7cef58227757e2673/transformed/sync15-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/fed1cac55c013d6371b66a402f3f2287/transformed/fxaclient-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c42b9c4e46b2b96c47285e03a83625cd/transformed/nimbus-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b1c6d02a26890dda1846fb2b57b62fc6/transformed/rustlog-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e9023d4f5f9625f15111f2b30a7b0555/transformed/support-rusthttp-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7048673b6f0d0b54b56e22a321a74eb3/transformed/httpconfig-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/be9ca1dd0727147ded6235d37dc7ed2b/transformed/full-megazord-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f66882ca041139e3dd81971d77936c3e/transformed/native-support-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3a0e9254adc3a4547ba6e0fd77500cc1/transformed/browser-domains-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/949c8747cacf87b857dada44e7ffbc7c/transformed/ui-widgets-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/39668193cf2c29dd165bbaffd9a77bdd/transformed/ui-colors-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b0db5f78a2d39e471700ad15a2a53405/transformed/ui-tooling-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/86b0a71d2a3c5b65881060a4c679c6f7/transformed/material-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9a30393d43acb816b35de980cc10635e/transformed/material-ripple-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c2ad0a3c7c4acd7f3628c900c45d7676/transformed/foundation-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c7169241fff195428b4383df7af904c1/transformed/ui-tooling-data-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/13bb243931ba55ac7025ec7de3a66c30/transformed/activity-compose-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d0755e08029f00c1f5834073deac4392/transformed/animation-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/33635c92682efcb1b0d71e62c35e094f/transformed/foundation-layout-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a5c310641e152f921d3c451efee01543/transformed/animation-core-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3b388578844252cc1593d616be3f274d/transformed/material-icons-core-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8097edcfe0a0bb39fb8088f357f86400/transformed/ui-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5039c9b5c9dd162b9f5b5e5fe94bd64f/transformed/glean-41.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/43d6028aa6312f58718c416613958988/transformed/work-runtime-ktx-2.5.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/87a8f2b378bc769a71cdb1f49fa3d635/transformed/room-ktx-2.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8c3fb3f5cc7b47eb058904f6695b1c65/transformed/ui-text-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9b14c60d083285ce8ff27b050d779ed8/transformed/runtime-saveable-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/4072c15d001370683286c271341865ce/transformed/ui-graphics-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/581c1289c528e1f9066d915a1c00102f/transformed/ui-unit-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/299b2cfbe32cac605c184e893423dcdc/transformed/ui-geometry-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ece49830db201e3dd84b1810bb83290c/transformed/ui-tooling-preview-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8c2a1b3ec2677ceddf18fc2a58ec2e5d/transformed/runtime-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3cdf9e5e246f63158bcf278d7e0c3fd8/transformed/preference-ktx-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/98f1441f5369b3098c93c4f555cd8d9d/transformed/navigation-fragment-ktx-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/83709bec1b283d4588b9f6d9fbe734ef/transformed/fragment-ktx-1.3.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/982a087c2f1e249ce4c80742e15fae49/transformed/navigation-runtime-ktx-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c45676d0e26dd3b6d78ee2a1c1da7169/transformed/activity-ktx-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0a00ee34c1873160352b6e160a35266a/transformed/lifecycle-viewmodel-ktx-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f4ea583d619b44e65e6b68711eb164f9/transformed/lifecycle-runtime-ktx-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.5.0/c1df91b076cf9a1af564bb7edd41c49683169e25/kotlinx-coroutines-android-1.5.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-debug/1.5.0/1406a87f78bfe3173af2b00d2710b053fb7ea77f/kotlinx-coroutines-debug-1.5.0.jar, file:/home/runner/.gradle/caches/transforms-3/9cdbda04e3169db96e2138814d95b632/transformed/lifecycle-livedata-ktx-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core-jvm/1.5.0/d8cebccdcddd029022aa8646a5a953ff88b13ac8/kotlinx-coroutines-core-jvm-1.5.0.jar, file:/home/runner/.gradle/caches/transforms-3/8d171355db3f1a761d7fb1ff451904db/transformed/geckoview-nightly-omni-94.0.20210930095024-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.5.20/636c5653641cd956de9aee5792155b07ea49825e/kotlin-stdlib-jdk8-1.5.20.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.5.20/218b60e1d446d1e0a18bc7aa8663634b136fbcc5/kotlin-stdlib-jdk7-1.5.20.jar, file:/home/runner/.gradle/caches/transforms-3/e3d23334357ca594c813df22bdeb708e/transformed/constraintlayout-2.0.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ed2f5352e9f86436513b3a7603b5a532/transformed/biometric-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b43d916e40e98cce64c5e0f3e6398ac0/transformed/navigation-ui-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9a396a2fcb9b0e04039edac7a6af15c9/transformed/material-1.2.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d9f9770bab99e9b087f2204dc7af36ad/transformed/preference-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7df94cb15a1b31b3cb97d29d1131cdc1/transformed/appcompat-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/36f4a612e6463f4ad2d8b1062f539028/transformed/legacy-support-v4-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f7e1dec922f964372b1b804f74dab8d8/transformed/legacy-support-core-ui-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3c61b90a234c04039b2551b8e2e69db5/transformed/coordinatorlayout-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.sentry/sentry-android/1.7.10/f70c4e3611686df8f5c1eda094313257418b135a/sentry-android-1.7.10.jar, file:/home/runner/.gradle/caches/transforms-3/3fee1c223c37de45d68994686c8afba7/transformed/ui-icons-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f154bd74cb54ca2cee9d8e51806361dc/transformed/paging-runtime-ktx-2.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/42dd6153ce6e7f70bd4db44c68d9f9dd/transformed/paging-runtime-2.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a06a4eebee0624c5dfa521af5baa5bea/transformed/viewpager2-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/38b5937f231c9174cca1d2285e06b660/transformed/recyclerview-1.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common-java8/2.3.0/cd3478503da69b1a7e0319bd2d1389943db9b364/lifecycle-common-java8-2.3.0.jar, file:/home/runner/.gradle/caches/transforms-3/f8364a66f80d9061959251ab7caa03f6/transformed/lifecycle-extensions-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1df0f70a1af1405246dda0d95623ba94/transformed/lifecycle-process-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0ab83552e28d378cb3ce60e140dd4fb6/transformed/navigation-common-ktx-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/26a223b8244bc65be1e01eb71d2be81e/transformed/core-ktx-1.3.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5edb318c6ecbdc1ba00b06ec8ff457e3/transformed/transition-1.4.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/63f7fe2142c454a7f67036cc6f5385b9/transformed/navigation-fragment-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d3f012e79571635d0e1f5c80395c0957/transformed/play-services-ads-identifier-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b95175af4adc71285a193ea421205429/transformed/firebase-messaging-20.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a1bc4a9e962f0d7b4bd392fa4a104829/transformed/firebase-iid-20.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3933de80dce0189a935fa61ada021f95/transformed/play-services-cloud-messaging-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5eb2072d6b3cea045411e65e98838c2d/transformed/play-services-stats-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6c9c3d503e76924b073d9712babb934b/transformed/firebase-datatransport-17.0.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ec37a2b5c487f411271c65243e695109/transformed/firebase-installations-16.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1f9f2db55779f3ebdfa681d0ec613712/transformed/firebase-common-19.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7d2fe8e311b405d00adf4423565f214d/transformed/firebase-installations-interop-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/daceeac7767e7efa7837c97519184087/transformed/firebase-iid-interop-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b471ced3bb768a7adc33a48e0c6b2378/transformed/play-services-fido-18.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a4ba4c8bafc8b4dd2b266ee4ed5619ae/transformed/play-services-base-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/30f0bbc6df9760c9d0316f1b9b83f4ef/transformed/play-services-tasks-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a176e95c95a09e897c5d032363de48ee/transformed/firebase-measurement-connector-18.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/30c5b9dae0efda29c2c74a55c1bead39/transformed/play-services-basement-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/09e9cece8e01c5ba1cf494eecb987981/transformed/fragment-1.3.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/fed3ed57d186ea15a3dc8408d950b16b/transformed/appcompat-resources-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/150fd957b52af41a0ebdc23195c8e963/transformed/browser-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/79aa8f50b8d555715ad2234e821b2be5/transformed/media-1.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f173018db664a1a1a5dc45bda939f61f/transformed/swiperefreshlayout-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f7d8923619223099f0b46d474f80a6eb/transformed/autofill-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a7889331eccbf5f0c19a288276649871/transformed/palette-ktx-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3380adb75e708561dd8ad877e5d05026/transformed/palette-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/903326348dd0f99acb1dc89f0cd866e9/transformed/legacy-support-core-utils-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3914dfdddf4ce77ae6338a9dee804ef3/transformed/drawerlayout-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1648b3c23b7cacbcb00db10e68b94ee7/transformed/viewpager-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7b7e0edaa9a4f8c4909fc7e527a3a6b4/transformed/slidingpanelayout-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8bc778553044aac12c4199eb654a10ac/transformed/customview-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9a1abe271afc7bf6929561d3e145d079/transformed/work-runtime-2.5.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/959e9da2e52ab63e586bab5a9f3b8811/transformed/vectordrawable-animated-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/07e596b24e5d431a3c6ab887f48a85a1/transformed/vectordrawable-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/580649d2970d5ae51beb4eee062a6a16/transformed/loader-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/73fd4a768136cf8de32a3fa3e7c3189d/transformed/asynclayoutinflater-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/42e9b060475236c104f1e93451a1416c/transformed/navigation-runtime-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/71efa7d87055b7ae23827ccf51edbda8/transformed/navigation-common-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9540726bd958458e5239bb399a4e78ce/transformed/activity-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1d026474ab1e44e37dd4e792bb62843c/transformed/core-1.5.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.adjust.sdk/adjust-android/4.18.3/af89b4dc961127355d7072b1423653b64a983154/adjust-android-4.18.3.jar, file:/home/runner/.gradle/caches/transforms-3/4fbc51a78232e43b61c8fb3e9a4ddb4b/transformed/installreferrer-1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7ae05f4498bbba343754926d9d2c3866/transformed/core-1.8.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c27602bf84caf3d360f125296d933ad2/transformed/falcon-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-agent-jvm/1.12.0/96421a1df31560296f7879a15953c5ef8637b049/mockk-agent-jvm-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-dsl-jvm/1.12.0/eb517db1a4dea71a3bdce5065b3b280b559ff958/mockk-dsl-jvm-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.5.10/d6c70f3c0df2457ea0095c61c1fc1188017dc3bc/kotlin-reflect-1.5.10.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-common/1.12.0/3d3fd6c65c1f6aa66210b8b5f581097d6a0d5e0e/mockk-common-1.12.0.jar, file:/home/runner/.gradle/caches/transforms-3/0266a6c4e0fe5f6738279cdbc40dc702/transformed/leakcanary-android-core-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a71ad76907b1a39fcc369e0f701e1eda/transformed/concept-fetch-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c7470aabd6845643e880878d8d6fa890/transformed/ui-util-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.paging/paging-common-ktx/2.1.2/3acd13bd21e696857fe39989070c1018ec79166a/paging-common-ktx-2.1.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.collection/collection-ktx/1.1.0/f807b2f366f7b75142a67d2f3c10031065b5168/collection-ktx-1.1.0.jar, file:/home/runner/.gradle/caches/transforms-3/47a36326d973acf0fd654fb55ef9d759/transformed/lifecycle-livedata-core-ktx-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/529627f676985c0f763ccdc3b422c82b/transformed/savedstate-ktx-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-agent-common/1.12.0/bbca73456f85b41b03b6d11414ebbf19360848a4/mockk-agent-common-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-agent-api/1.12.0/415fd3961ba729257ddfe6639be6124869335ca8/mockk-agent-api-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-dsl/1.12.0/abe7c99dd6b8a2b8d2e10c2b103d36775c10b8a9/mockk-dsl-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-android/2.4/d07c172a763fbfd0e9450f1ba1480ce4c633fdd3/shark-android-2.4.jar, file:/home/runner/.gradle/caches/transforms-3/0da74ab4a834cbab501f7d01b43b45fb/transformed/leakcanary-object-watcher-android-androidx-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ab70d29508b64e9aa1c12c0988bf2892/transformed/leakcanary-object-watcher-android-support-fragments-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d4fa3074a9779db7fab01f7bdff2f77e/transformed/leakcanary-object-watcher-android-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1957cccd6dc45a1124d02fe4fcc058be/transformed/plumber-android-2.4-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.5.20/9a565217f3ba7b873ffe0617d0c681929d82d893/kotlin-android-extensions-runtime-1.5.20.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark/2.4/5aa97ac9e71a6127388e13a8ea070586f41f4014/shark-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/leakcanary-object-watcher/2.4/a40f33d9b1c5218e6da063dc996c89d0dac122d1/leakcanary-object-watcher-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-graph/2.4/89557baa9c59c4bf0e667905942966f3513d5cf6/shark-graph-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-hprof/2.4/367ac7b4f37a64747fcf5327258cd5f82496bf89/shark-hprof-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-log/2.4/92aa5a581103961c3b56b31e27bb349def942498/shark-log-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/2.2.2/36f483536153f15339a8b48d508e22be7c9c531a/okio-2.2.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.5.20/9de35cc611bcecec8edce1d56d8e659953806751/kotlin-stdlib-1.5.20.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/3.11.2/bb97012477172d7c04ff2193e31fcb38dd171879/mockito-core-3.11.2.jar, file:/home/runner/.gradle/caches/transforms-3/af2185c3b044ab6839bf78bd9be3b301/transformed/core-ktx-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-project/2.2.1/8239e98c16f641d55a4ad0e0bab0aee3aff8933f/maven-project-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-artifact-manager/2.2.1/ec355b913c34d37080810f98e3f51abecbe1572b/maven-artifact-manager-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-error-diagnostics/2.2.1/e81bb342d7d172f23d108dc8fa979a1facdcde8e/maven-error-diagnostics-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-settings/2.2.1/2236ffe71fa5f78ce42b0f5fc22c54ed45f14294/maven-settings-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-profile/2.2.1/3950071587027e5086e9c395574a60650c432738/maven-profile-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-plugin-registry/2.2.1/72a24b7775649af78f3986b5aa7eb354b9674cfd/maven-plugin-registry-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.plexus/plexus-container-default/1.0-alpha-9-stable-1/94aea3010e250a334d9dab7f591114cd6c767458/plexus-container-default-1.0-alpha-9-stable-1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.12/2973d150c0dc1fefe998f834810d68f278ea58ec/junit-4.12.jar, file:/home/runner/.gradle/caches/transforms-3/bc2a85f52e4b03241a838d0978a4fa95/transformed/core-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.6.1/f434e4745c28ec2e75ca58ab81fa0678c4c6be74/shadows-framework-4.6.1.jar, file:/home/runner/.gradle/caches/transforms-3/518de95027b90bd01ce639d2928ddf70/transformed/monitor-1.4.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1296e7608087561c216bd8455fd8822f/transformed/cursoradapter-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/cb2a9375ad8202f00eae50b1c0d91eac/transformed/lifecycle-viewmodel-savedstate-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1e6ebf045bc5881e77c8afc03e30d96c/transformed/savedstate-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8ef45442972f965c609c68b3508c82e4/transformed/lifecycle-service-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/db10c13ff81eb2de59589a435e97b66c/transformed/lifecycle-runtime-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/45dd3aeb96a1f09d97c80c5a177d00c6/transformed/lifecycle-viewmodel-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/386fd8e4f40b82b438ba4fc30baee74a/transformed/cardview-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6e72a666f90f1cc366b5be76a10cdd89/transformed/localbroadcastmanager-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/2559d1d917a3c0e65caf6c14e8904afc/transformed/profileinstaller-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e65e725e9ac47184a44ff7ba5c712eb1/transformed/versionedparcelable-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.paging/paging-common/2.1.2/e8cdd44767173c4af5164732c32741c89afa684d/paging-common-2.1.2.jar, file:/home/runner/.gradle/caches/transforms-3/0c3052771365bfed11824b32a5339905/transformed/startup-runtime-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7746c89b92c26c5a20c44f13a8dffef7/transformed/tracing-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b57871dbd59e51561b9af525796a385f/transformed/lifecycle-livedata-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/74544aac2b3ab49ec5c1b881137ce2ec/transformed/lifecycle-livedata-core-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/499d422fc2e1ff5ffc5ea21b644320ef/transformed/room-runtime-2.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/73be4a49f8025a97bbbd42e8ce9eb8c6/transformed/core-runtime-2.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.3.1/fc466261d52f4433863642fb40d12441ae274a98/lifecycle-common-2.3.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.concurrent/concurrent-futures/1.0.0/c1e77e3ee6f4643b77496a1ddf7a2eef1aefdaa1/concurrent-futures-1.0.0.jar, file:/home/runner/.gradle/caches/transforms-3/efe3f3ee1dc30a1c9cd84271b15e271a/transformed/interpolator-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.3.0/11e5b5194cb53b2c5ebfa57710a1a74c67cb3335/room-common-2.3.0.jar, file:/home/runner/.gradle/caches/transforms-3/34e6d0d4aa93b1685d645dd28b22bb6f/transformed/transport-backend-cct-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7c5000aa5c1863c933bec5e4c6806c98/transformed/transport-runtime-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/18e628fc0f7b6044b72919839e900ec4/transformed/firebase-components-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/19b6e592012085f30ca322a3e3a5815b/transformed/firebase-encoders-json-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/273c27a557d721002b28ffb800682fcb/transformed/documentfile-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/bbd96c7293056e8b5ac41a4087c28f7f/transformed/print-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9bc90ff6346a510ed11e151ba9ad693b/transformed/sqlite-framework-2.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1dd29afe92de61dc44220398bcc6f41f/transformed/sqlite-2.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.2.0/57136ff68ee784c6e19db34ed4a175338fadfde1/annotation-1.2.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.6.1/4332f14f797763028dbcec462b7956a507320690/junit-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.6.1/444de1a4b500db4531bc355c395a8cf4655328a4/resources-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.6.1/6f07458cd93eb90a68679d9f64b9fc7057890f4b/sandbox-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.6.1/409d6c0557f8f2a59782483a5813ac50b75bde54/utils-reflector-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.6.1/b16422d286873448f3deb8737657c2aff876b546/plugins-maven-dependency-resolver-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.6.1/2df9b53fac09615e6bc0c77ee36b8483a05ee825/shadowapi-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.6.1/d14910e168c8a931fde766e64b49fc5d3f1f7976/utils-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.6.1/95d8b0a2afb69c8de1e56abc02728e09cf60bf25/pluginapi-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.6.1/246a0f0bd16253b7d51e0bf2ef3646bf38acdbc5/annotations-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.dagger/dagger/2.24/97f35169621dd6a30d5958e8326e30f76ebe2e62/dagger-2.24.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.68/46a080368d38b428d237a59458f9bc915222894d/bcprov-jdk15on-1.68.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.ant/ant/1.8.0/7b456ca6b93900f96e58cc8371f03d90a9c1c8d1/ant-1.8.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/classworlds/classworlds/1.1-alpha-2/5adf2e681c57d7f48038b602f3ca2254ee82d47/classworlds-1.1-alpha-2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-artifact/2.2.1/23600f790d4dab2cb965419eaa982e3e84c428f8/maven-artifact-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-model/2.2.1/c0a1c17436ec3ff5a56207c031d82277b4250a29/maven-model-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-file/1.0-beta-6/6c53633505460caf49d2660de1e24744d915afb9/wagon-file-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-http-lightweight/1.0-beta-6/b3815078570c3b1f0667e123d59717c6b726c6c2/wagon-http-lightweight-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-http-shared/1.0-beta-6/ccd70d7e0d8c085e648a83f072da06ca9a53be94/wagon-http-shared-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-provider-api/1.0-beta-6/3f952e0282ae77ae59851d96bb18015e520b6208/wagon-provider-api-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-repository-metadata/2.2.1/98f0c07fcf1eeb213bef8d9316a9935184084b06/maven-repository-metadata-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.plexus/plexus-utils/1.5.15/c689598ce1eb94c304817877ed15911099972526/plexus-utils-1.5.15.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.plexus/plexus-interpolation/1.11/ad9dddff6043194904ad1d2c00ff1d003c3915f7/plexus-interpolation-1.11.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna-platform/5.5.0/af38e7c4d0fc73c23ecd785443705bfdee5b90bf/jna-platform-5.5.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna/5.8.0/3551d8d827e54858214107541d3aff9c615cb615/jna-5.8.0.jar, file:/home/runner/.gradle/caches/transforms-3/5d30a58e1dc4d81c0ebc75d9c1c4bcd9/transformed/jna-5.8.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.constraintlayout/constraintlayout-solver/2.0.4/1f001d7db280a89a6c26b26a66eb064bb6d5efeb/constraintlayout-solver-2.0.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.sentry/sentry/1.7.10/2a2fd502a52f9f067b6f34f3111aff7803b66d25/sentry-1.7.10.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.jakewharton/disklrucache/2.0.2/93b399470e72c0bc8cb52379943b63c9aa586cd8/disklrucache-2.0.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.zxing/core/3.3.0/73c49077166faa4c3c0059c5f583d1d7bd1475fe/core-3.3.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.5.20/3d79dbd48bf605f4aac1e7028981a1953e245cbb/kotlin-stdlib-common-1.5.20.jar, file:/home/runner/.gradle/caches/transforms-3/81c8199effec3759afc59c9924126454/transformed/annotation-experimental-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.11.5/3145358bfab3d86aebac5f31c7703505ebb108c1/byte-buddy-1.11.5.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.11.5/ec760e507143fb9696595150a03573b066f46887/byte-buddy-agent-1.11.5.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/3.2/7fadf57620c8b8abdf7519533e5527367cb51f09/objenesis-3.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/27.0.1-jre/bd41a290787b5301e63929676d792c507bbc00ae/guava-27.0.1-jre.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/9.0/5a34a3a9ac44f362f35d1b27932380b0031a3334/asm-commons-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/9.0/7c059a94ab5eed3347bf954e27fab58e52968848/asm-util-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/9.0/4630afefbb43939c739445dde0af1a5729a0fb4e/asm-analysis-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/9.0/9df939f25c556b0c7efe00701d47e77a49837f24/asm-tree-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/9.0/af582ff60bc567c42d931500c3fdc20e0141ddf9/asm-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/1.0.392/d6234e08ff4e1607ff5321da2579571f05ff778d/sqlite4java-1.0.392.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.7.4/eff48ed53995db2dadf0456426cc1f8700136f86/auto-value-annotations-1.7.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.ant/ant-launcher/1.8.0/8b53ba16fa62fb1034da8f1de200ddc407c8381/ant-launcher-1.8.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/backport-util-concurrent/backport-util-concurrent/3.1/682f7ac17fed79e92f8e87d8455192b63376347b/backport-util-concurrent-3.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.24/1da441001b817bc03a53e6393f2014e59e1438a/snakeyaml-1.24-android.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.24/3f6b4bd4f8dbe8d4bea06d107a3826469b85c3e9/slf4j-api-1.7.24.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.8.7/8b46f39c78476fb848c81a49fa807a9e9506dddd/jackson-core-2.8.7.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protobuf-javalite/3.11.4/e3b3f442b01f8fa9bb54893abc44aa352c9b1afd/protobuf-javalite-3.11.4.jar, file:/home/runner/.gradle/caches/transforms-3/72bfcb85c4f3dc5da9f607c6de4f79da/transformed/transport-api-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/2.5.2/cea74543d5904a30861a61b4643a5f2bb372efc4/checker-qual-2.5.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.2.0/88e3c593e9b3586e1c6177f89267da6fc6986f0c/error_prone_annotations-2.2.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.1/ed28ded51a8b1c6b112568def5f4b455e6809019/j2objc-annotations-1.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.17/f97ce6decaea32b36101e37979f8b647f00681fb/animal-sniffer-annotations-1.17.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/nekohtml/xercesMinimal/1.9.6.2/d1c5e063683a0e6f77cd5f051a9d4af48346fa6/xercesMinimal-1.9.6.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/nekohtml/nekohtml/1.9.6.2/2d960be7b62ae6622dbbbe49ab4ffdc609f85c80/nekohtml-1.9.6.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/java_res/debug/out, file:/home/runner/.gradle/caches/transforms-3/ba191990df0f6ee7059779aaa5f3ab6f/transformed/support-test-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7ca3872a669cf94ba4aabdfe019cb0bd/transformed/support-test-libstate-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/42fa9d0501667c17c5b01ba40d48e4a7/transformed/junit-ktx-1.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/db22c41db31da5b8807029623ef7bd29/transformed/junit-1.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/380f889f9f31dd996ce5098eff96134a/transformed/work-testing-2.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7dbceeb74d0b10830e9cf8aae06f05d5/transformed/feature-pwa-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/093d8a56d66dd9ec95459678bb72aa7f/transformed/feature-customtabs-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1ff4447287b325c66dfec661addea073/transformed/browser-toolbar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2326b42998c8dbebf1865d32533ef6fb/transformed/feature-readerview-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/759ba40053b4495804d4b30507a32800/transformed/browser-menu-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c140eff10d6deac8dad236130a4e1dcb/transformed/support-migration-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/43580059c4d6da694f065767b65f70ad/transformed/feature-addons-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0a49382d4e4bca7d07c7c86dda97af92/transformed/feature-downloads-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/15c11f3bf762162978a5237479999ccd/transformed/compose-awesomebar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/77fc7440ccc4f4e6cf81285b336c8330/transformed/feature-awesomebar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2bc01451e492f9aa47b9ba562e3aad6f/transformed/feature-syncedtabs-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c634479160fd1f87b909e269841af291/transformed/concept-awesomebar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/27013ace9d93b5c35f3b9ba27d781b47/transformed/browser-engine-gecko-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0825e5ded0cd3cf326e5df75be59d5a2/transformed/browser-tabstray-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/14da3df9b97111880cb76a221477a548/transformed/feature-accounts-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20382eb9488703791ef9d36fe646529c/transformed/feature-contextmenu-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/987368d91f4b57017323d25e8fd00373/transformed/feature-intent-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0cea274b6d8031572389a38b8f5ed1f6/transformed/feature-search-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/42c89d3ffcc53783754e7a771f9bc1f4/transformed/feature-tab-collections-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e2ccd78d5e14a477f4f02a1973599a4b/transformed/feature-tabs-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/15f71e649444277f48b59e9159ada092/transformed/concept-tabstray-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7d8fab755d1a1f239c5a84d1eaa34095/transformed/feature-webnotifications-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0a9a33ec51107b80ef499fd0140c2078/transformed/browser-icons-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/54071b7a3eb211f882d05453e225bdb5/transformed/browser-session-storage-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e49f6c17701941bfbc4f376be7750138/transformed/browser-thumbnails-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/db74f0b6c93e6559a184005e94a655a8/transformed/feature-app-links-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2af7b54513f20009493e5261f4e68753/transformed/feature-media-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d4bfa18126161bf331d2f37552fe3412/transformed/feature-logins-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4da75c1f9005cdc0ae9481d1fdb3dd11/transformed/feature-prompts-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bea34efdad5f410ba63d5cb6249b7ab9/transformed/feature-privatemode-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b94de645380f13afee5e46b7247bb199/transformed/feature-toolbar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5cdda9636b56247af2be75a55c1c15cd/transformed/feature-session-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f8c9f8a8ca6e555c0d58505cb16a95c3/transformed/feature-findinpage-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/94ddcc084014b87a316f061ecfb0e672/transformed/feature-sitepermissions-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/32cf17543c6631dcd9aaf06b802046a6/transformed/feature-recentlyclosed-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/668ac8f2d4281349215e37f9f635251f/transformed/feature-webcompat-reporter-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/69aeb71e52c2cfb1c5bfb0030f6a9335/transformed/support-webextensions-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fe0b54d3ddba6bba80833cda32a3298d/transformed/service-nimbus-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5a79bbfb1ee7bc8016ceadf7d2c3768c/transformed/support-locale-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/69bd7e65580e13ce3a38c1315e3cfb79/transformed/browser-state-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0fd4593ef384c47dd4527056629e3fe1/transformed/feature-webauthn-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/01e3ed1f6937932128eea00820cdcd1a/transformed/feature-webcompat-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/71ca13a14f81740e453cb70bafde7e8c/transformed/concept-engine-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/36f77a54927d1b9c5ceeb2d6570a2685/transformed/ui-tabcounter-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d75e8fd2555d845de178b9853bea1628/transformed/browser-menu2-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f1391e46b82714b92cdf4f6931decf5e/transformed/concept-menu-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/39d2fd2db1d611d20dcec7eb3bd47dd4/transformed/feature-accounts-push-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d8754f401e2dc6696412dbec224722dc/transformed/feature-push-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1a86558ca2761a9ab8dad55b9e7cc858/transformed/lib-push-firebase-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e74ffae6b8af80fa56d22340e090196e/transformed/concept-push-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5c32d4574c1c0fabf62c7060a4ae950e/transformed/feature-top-sites-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e1c58271a6850edb5cba7b3890712452/transformed/browser-storage-sync-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/71af42e4e8705719794bfd9db35800d0/transformed/service-sync-autofill-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/21a2593e8602564c2d944781d5ab2f57/transformed/service-sync-logins-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1d66762d16d191a821953258d29c6f38/transformed/service-firefox-accounts-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/19a44b067d8eb375611f229bd17bb341/transformed/concept-sync-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/83835604b0c17bbba06b4b9c982564c4/transformed/concept-toolbar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/01a6a132e42a947026a5d3d07931e31e/transformed/feature-autofill-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/13c9e3533dd67fa4ab4a136c692353e8/transformed/feature-qr-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9db59d32f5f44a7e261cbb34f70d7459/transformed/feature-share-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20f3cea81f89cdd505ef1fbbb9443fc9/transformed/service-pocket-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b3b39bf157a71d54061591dae020bc07/transformed/service-digitalassetlinks-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d28be1f1fa24299b84b6bfe0b48fe608/transformed/support-sync-telemetry-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d4e65b6eacbbffa4b09a7ced543af578/transformed/service-glean-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fdbcb818d21cfadb5101cefd38d83d9f/transformed/service-location-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/366e806dd7a3416b1482b329f1fd4bb3/transformed/support-images-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/70e1aac7f4cd159d14cccfc3d2f551a9/transformed/lib-publicsuffixlist-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/de6f972524db9978bf0af70f1e2cff1b/transformed/lib-crash-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5aa67e3c78bfec285ab46fb82bbcc0ed/transformed/lib-state-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bfc9517a6467b29b8b94b88b3bcc6364/transformed/browser-errorpages-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b0a48eaeee227438fdd980d4afee15a2/transformed/support-ktx-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/07c21d906d6b2dffa4eaf539286c024e/transformed/support-rustlog-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c96eb0cef4ce3476cfb785475f2215f8/transformed/ui-autocomplete-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/05ed3cdb893628e006c57b4b9deb6140/transformed/support-utils-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ed7c22196613f9818519e173ce026472/transformed/lib-dataprotect-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d1cb066697fb607634f1214d875aaef5/transformed/support-base-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b82cb7a9ee82241031d41d1a85839609/transformed/viewbinding-7.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2aef12d5a7eae920c520d689d485f900/transformed/leakcanary-android-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ed063d68580f8c41f2d0a5ddf12c1312/transformed/concept-base-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0c3a98692d77145bbeba0b4a138b9f5a/transformed/concept-storage-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/8a7a87ab86002820e06d01f01ccc7bea/transformed/places-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ab724b887f2b12015f02275ad2900ad0/transformed/tabs-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4811137968708d328838823dcc4cbf09/transformed/push-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ece57c3a8f016c30da4ed75f1e187c15/transformed/autofill-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ed1372b3267067e9f943b5f6e08fcea6/transformed/logins-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20d17ce1c1839670a8abe744c163d1bf/transformed/syncmanager-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bdc5d20032f3a2bc5a6cb233879b04b4/transformed/sync15-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/cd5f27c89abc06789b2200d682aa45e3/transformed/fxaclient-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5371440af6a0fb710dc568c7adfbe9c3/transformed/nimbus-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b9f83d0b8df553d24d15d36f6e35b8d3/transformed/rustlog-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4f09cc39fa145db62fc7c48921763456/transformed/support-rusthttp-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/60f433cddd20f846621020fd82e15275/transformed/httpconfig-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d89c11a086e1c372739b5237e6041eb4/transformed/full-megazord-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/297451f1989f239923049c16a3e4670d/transformed/native-support-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e531b0bc3a99065ac3c8fdd51403d1a3/transformed/browser-domains-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/baf9825fc08199deae594085033f277d/transformed/ui-widgets-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/811277e4ea26343b3293d26612b7482c/transformed/ui-colors-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/8f38f873c37b9d5c42d6293ae7082cdb/transformed/ui-tooling-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e1d9434246f02fda28323fb79c693000/transformed/material-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/07423513a63e491c6d9ae9ce9ee6840e/transformed/material-ripple-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ccc2cb572135c1abf4b88c2598a7e618/transformed/foundation-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e58f01de349caecfdeb7798ec4dc83c5/transformed/ui-tooling-data-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a01a4f4f554bedf668b35814fa96845a/transformed/activity-compose-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/05d5a3f014fe04a0fe51a3de9b941fae/transformed/animation-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d3c3a817f279394983587c9d9da69e92/transformed/foundation-layout-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e1c7ce8fad4a5fdf7a87f2b6eb533071/transformed/animation-core-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3e48c0cf5911f82561e38f77bb6afa4d/transformed/material-icons-core-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/abe560afde1d89f3f66600291f758bc5/transformed/ui-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e47b09aeaca63b282cc110eba5ebb8d9/transformed/glean-41.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e993a2e5b27084fc183d8ade17c4d3c6/transformed/work-runtime-ktx-2.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4dc3ffb354b95ab4f58ac3dca3dc06ae/transformed/room-ktx-2.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/73f6caa643aa0dfe24fbd443e107dab3/transformed/ui-text-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/853de0dda20803883bd992c4674ef8cd/transformed/runtime-saveable-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/91f27f0fa2633bbb79ef6df7e3561e1c/transformed/ui-graphics-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1ccfb84cfe130eb88f52c2b7e088cb10/transformed/ui-unit-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/30d4f7e60955de59b3f27ebbf2e3ec06/transformed/ui-geometry-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6238f88043642cc0c4acd7b8a7ec04c2/transformed/ui-tooling-preview-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4d3b9344c0e1f2f1418b25fc2ae12865/transformed/runtime-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/603606fccb1c344df8ee7c76ffeeb16b/transformed/preference-ktx-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a72ed687c79c4c54fe0d9b299d21db45/transformed/navigation-fragment-ktx-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a87b0a3a8a87252e15cd837f7f0185d0/transformed/fragment-ktx-1.3.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/afa77158e0883c7b9e89899a1a913e87/transformed/navigation-runtime-ktx-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1bfc24c8c978434d0b885d2467faf54e/transformed/activity-ktx-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6e40a7e0dd995dd5674680bbcd874d26/transformed/lifecycle-viewmodel-ktx-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4a10a829855e083dbcdef8f1ce95ed1e/transformed/lifecycle-runtime-ktx-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bf223090195bea59f6ea2b85e3f4345b/transformed/lifecycle-livedata-ktx-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9dbc490c6adfaf3d68a8ebfa41b8eadf/transformed/geckoview-nightly-omni-94.0.20210930095024/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/115366cfb9f40c21fe292087bc3f790d/transformed/constraintlayout-2.0.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7b4bd1add714033a504aaf9cb031fb68/transformed/biometric-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6ba5a61902f74993792f2cb8f7f4e098/transformed/navigation-ui-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/26d37146b3c97d6a500d5530d371e156/transformed/material-1.2.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a4bd908a89b20eff8a6571dd9a83266d/transformed/preference-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9422d6bb91cfaafbfb5bc59f84755587/transformed/appcompat-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/412946f166f3f67f1feaf3dc46fa7529/transformed/legacy-support-v4-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1f3cb2469a8de97c51aed231687f2615/transformed/legacy-support-core-ui-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/67e7e04105ee2d03cfd4da1706e1da26/transformed/coordinatorlayout-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/91ca8fece7e84b783f009b70aa6cd502/transformed/ui-icons-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e8e16c27d463e3c84075ae9ebef66578/transformed/paging-runtime-ktx-2.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/dece0ebfae7173b3c6d7e875e3a9bab1/transformed/paging-runtime-2.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fe77c872280f791e6047f22693fc4b91/transformed/viewpager2-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/645d044dba9de8a7b2f311f022cf9f83/transformed/recyclerview-1.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/44cdd0edc77e18fbf201ef53523f1bf5/transformed/lifecycle-extensions-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/53d1cd5baa65a8a895665a52fdfa23bb/transformed/lifecycle-process-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f72319d3153e6c45413e8b993e50deb2/transformed/navigation-common-ktx-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/907b04b48f4b93647e8374c4f0285856/transformed/core-ktx-1.3.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/edf13d1d369ec6bfd5935d204c8e7673/transformed/transition-1.4.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/15258e48ff5c33f4eeaeceb4449ad053/transformed/navigation-fragment-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/cf20fd632fe42d4671a6baf530553e78/transformed/play-services-ads-identifier-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3cc7f8e04a58dcfeb250b4b7be67ed5d/transformed/firebase-messaging-20.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c6e7b11f6439f4682b0398779bc5d36b/transformed/firebase-iid-20.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/38043d5c5d74f3dec14e2efb5aba1627/transformed/play-services-cloud-messaging-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0e2e41c79d1a2e316fca3157720f16ce/transformed/play-services-stats-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6a35eb2dc17d6fced71898c45d791271/transformed/firebase-datatransport-17.0.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e0d54e7da2b3ed888bdeb91979edb04d/transformed/firebase-installations-16.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0fa057de802823f66bec878f7f0860f6/transformed/firebase-common-19.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4acad1ea353f1ecc18a9bf5cad79ccc5/transformed/firebase-installations-interop-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/276f51d8175ad34843894d268198d7d4/transformed/firebase-iid-interop-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4733c8b38f0608b3e89a306d651e891a/transformed/play-services-fido-18.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2e4a98621f35e8480a9d02c354300813/transformed/play-services-base-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d9845b0d6f08a83f2452d869cc8eed37/transformed/play-services-tasks-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e623264bc1342b3c7f31331900a99ce0/transformed/firebase-measurement-connector-18.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6e9c8cfc96123ff5d1b39da70338d3e9/transformed/play-services-basement-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/cfd0a9991ac3c01445e71f0c9b3eba33/transformed/fragment-1.3.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2a9902251d53c99075777af9526df232/transformed/appcompat-resources-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9be78be6b0065bbfb3105cad94e650b8/transformed/browser-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5fbca0f08b8cbc7878ee6c510488f24b/transformed/media-1.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2e719edbb7e80156414efa5eb136071e/transformed/swiperefreshlayout-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b58b3d9f183ce4cd3e53dbae00bf6ce3/transformed/autofill-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6fd5723aee2ba7e80f4994c5b849bc34/transformed/palette-ktx-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2de9a1667ef1a05c5994656b5803cf96/transformed/palette-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9e10e7e492ba3d404c9862cc250e017f/transformed/legacy-support-core-utils-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2afb91b68ef6cdf1acf1a36e923a81f9/transformed/drawerlayout-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4a6bca33619975143569ca1ff8c8f929/transformed/viewpager-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/99da73a775c1e60c447661fed4489635/transformed/slidingpanelayout-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0543789f3528b386f27a68cc17e37b2b/transformed/customview-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/edc0fd9c8e79a6d8dac1dc35816bbb45/transformed/work-runtime-2.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d85e9c74a87ec90781ff57ab6cfe337d/transformed/vectordrawable-animated-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e8b7308f355cb57695ab0c96b7ba690a/transformed/vectordrawable-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c2de62ea70f364e2b3d343b75de5b468/transformed/loader-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f1d8728800449b637ece4005eda06dd9/transformed/asynclayoutinflater-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/320801b521f0453c5fdeba3101d9f171/transformed/navigation-runtime-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/297ec4b74523ef68edfab79e849f2115/transformed/navigation-common-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5c4043fec5bd558006718aee0e4e903a/transformed/activity-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/baad857560f2d4ff00b8865d7ed727d6/transformed/core-1.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/24960fddf98576217b2bead3e11f5dcf/transformed/installreferrer-1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0bbd9961f62d2d50ab567131ef17110a/transformed/core-1.8.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7eff327a33e7987b551f1cf663275bdd/transformed/falcon-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9a315ffd33d6c9690e8340f55d1ebfe4/transformed/leakcanary-android-core-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ee3d2cbcd1e0e6a785cfbb459aa0db59/transformed/concept-fetch-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e14626c37a7f24c433c3195e0fa320e4/transformed/ui-util-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20a55625c06c6b37c5c8d249a712a72c/transformed/lifecycle-livedata-core-ktx-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a93f7da337ed6ee445829abe04db837f/transformed/savedstate-ktx-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/48e056d000755372688f06e73963ee68/transformed/leakcanary-object-watcher-android-androidx-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/eb9dd8cf6304e2a86ecfc62452698816/transformed/leakcanary-object-watcher-android-support-fragments-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/776fab5fbf36040166d8a7b9cae5793a/transformed/leakcanary-object-watcher-android-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2ad8da4aac8c5c06718078f5ae42ef68/transformed/plumber-android-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5b8b2935491cd32e7ef8f576ad11d97e/transformed/core-ktx-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/dc918cd378f4b0296779ed6753ca63d0/transformed/core-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/75e0f7015fabe5db37dd3c6514e3f4fb/transformed/monitor-1.4.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2ea47e1c6d6558ba78e3075decdd8e57/transformed/cursoradapter-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/356085e61d7c62f307433a2fd22100ac/transformed/lifecycle-viewmodel-savedstate-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/423546c57d677b30e65436d939c1b94a/transformed/savedstate-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/53e7545eef63429183cf5dcf4e852cd4/transformed/lifecycle-service-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b4c3aedb5fce9b1c10ea9983d3b076fe/transformed/lifecycle-runtime-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/451cb01dd8864e5aacdc2822d66e171b/transformed/lifecycle-viewmodel-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/28121fc8a4f55cd5c57a013f9ef91777/transformed/cardview-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3e87ba80ea84d86c3f2422265c87fa52/transformed/localbroadcastmanager-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a857a917ea921863d507ebf4ef2c7cd7/transformed/profileinstaller-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/df024029ffa7345980aaa2be52cdd9eb/transformed/versionedparcelable-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/660598e145725505a4069b3186f83fb4/transformed/startup-runtime-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5481c8b693c894066cdc4344331d59e7/transformed/tracing-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/60093caf4c9af68a5f11ee8ff08afada/transformed/lifecycle-livedata-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3bc690d5184972738662d02e2e236004/transformed/lifecycle-livedata-core-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/770a7de97ffdccd0122e08df65419d61/transformed/room-runtime-2.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fe9e2e4af469423ee74cf5f627f235f2/transformed/core-runtime-2.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c7e8ca8afda093085ca0399eb6bcdc64/transformed/interpolator-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/8b4401888f2b120c267061ce7afadd99/transformed/transport-backend-cct-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fc200096c4a4e1e717515066b2ed1de2/transformed/transport-runtime-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/66d8e689d4da23c6f7ff7c0ec057dacb/transformed/firebase-components-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5d34bc4bc1c38336014dec3258cb9209/transformed/firebase-encoders-json-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f7e664fe8b72b9ab5e485369273cec33/transformed/documentfile-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ad3b5cd9f2583fed60fe5cfa4e25a109/transformed/print-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b67b80f3768825d25e8c8cfa53304d87/transformed/sqlite-framework-2.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7eaef8dbfe59dec6c56a216baefca3ed/transformed/sqlite-2.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e06f9402bed5dca232cfed6ba191ffe1/transformed/jna-5.8.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5ec6a40beaa4bdc572d78c7aecbaf7c3/transformed/annotation-experimental-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/031dcc11309b63cd09fb5378a04aff8a/transformed/transport-api-2.2.0/jars/classes.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/debug/R.jar, file:/home/runner/.gradle/caches/transforms-3/1ebd8edeb059821eaa6de7c916635ea9/transformed/android.jar])
2021-10-01T02:51:33.5869607Z     at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:301)
2021-10-01T02:51:33.5871249Z     at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:461)
2021-10-01T02:51:33.5872506Z     at com.sun.jna.Library$Handler.<init>(Library.java:192)
2021-10-01T02:51:33.5873288Z     at com.sun.jna.Native.load(Native.java:596)
2021-10-01T02:51:33.5874146Z     at com.sun.jna.Native.load(Native.java:570)
2021-10-01T02:51:33.5875349Z     at org.mozilla.experiments.nimbus.internal._UniFFILib$Companion$INSTANCE$2.invoke(nimbus.kt:1759)
2021-10-01T02:51:33.5876949Z     at org.mozilla.experiments.nimbus.internal._UniFFILib$Companion$INSTANCE$2.invoke(nimbus.kt:800)
2021-10-01T02:51:33.5878357Z     at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
2021-10-01T02:51:33.5879876Z     at org.mozilla.experiments.nimbus.internal._UniFFILib$Companion.getINSTANCE$nimbus_release(nimbus.kt:800)
2021-10-01T02:51:33.5881978Z     at org.mozilla.experiments.nimbus.internal.NimbusClient.<init>(nimbus.kt:1583)
2021-10-01T02:51:33.5883742Z     at org.mozilla.experiments.nimbus.Nimbus.<init>(Nimbus.kt:343)
2021-10-01T02:51:33.5885055Z     at mozilla.components.service.nimbus.Nimbus.<init>(Nimbus.kt:55)
2021-10-01T02:51:33.5886438Z     at mozilla.components.service.nimbus.Nimbus.<init>(Nimbus.kt:49)
2021-10-01T02:51:33.5888378Z     at org.mozilla.fenix.experiments.NimbusSetupKt.createNimbus(NimbusSetup.kt:71)
2021-10-01T02:51:33.5890093Z     at org.mozilla.fenix.components.Analytics$experiments$2.invoke(Analytics.kt:105)
2021-10-01T02:51:33.5891885Z     at org.mozilla.fenix.components.Analytics$experiments$2.invoke(Analytics.kt:104)
2021-10-01T02:51:33.5893254Z     at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
2021-10-01T02:51:33.5895144Z     at org.mozilla.fenix.components.Analytics.getExperiments(Analytics.kt:104)
2021-10-01T02:51:33.5897095Z     at org.mozilla.fenix.FenixApplication.setPreferenceMetrics(FenixApplication.kt:695)
2021-10-01T02:51:33.5899348Z     at org.mozilla.fenix.FenixApplication.setStartupMetrics$app_debug(FenixApplication.kt:546)
2021-10-01T02:51:33.5982323Z     at org.mozilla.fenix.FenixApplicationTest.WHEN setStartupMetrics is called THEN sets some base metrics(FenixApplicationTest.kt:127)
2021-10-01T02:51:33.5984645Z     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-10-01T02:51:33.5986959Z     at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-10-01T02:51:33.5989627Z     at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-10-01T02:51:33.5991621Z     at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2021-10-01T02:51:33.5993080Z     at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
2021-10-01T02:51:33.5995113Z     at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2021-10-01T02:51:33.5997447Z     at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
2021-10-01T02:51:33.5999829Z     at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2021-10-01T02:51:33.6002129Z     at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2021-10-01T02:51:33.6003806Z     at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
2021-10-01T02:51:33.6004952Z     at org.junit.rules.RunRules.evaluate(RunRules.java:20)
2021-10-01T02:51:33.6006524Z     at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:570)
2021-10-01T02:51:33.6008560Z     at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$0(SandboxTestRunner.java:278)
2021-10-01T02:51:33.6010395Z     at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:89)
2021-10-01T02:51:33.6012222Z     at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2021-10-01T02:51:33.6014161Z     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-10-01T02:51:33.6016523Z     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-10-01T02:51:33.6017853Z     at java.base/java.lang.Thread.run(Thread.java:829)
2021-10-01T02:51:33.6019087Z     Suppressed: java.lang.UnsatisfiedLinkError: libmegazord.so: cannot open shared object file: No such file or directory
2021-10-01T02:51:33.6020289Z     at com.sun.jna.Native.open(Native Method)
2021-10-01T02:51:33.6021374Z     at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:191)
2021-10-01T02:51:33.6038451Z     ... 38 more
2021-10-01T02:51:33.6039480Z     Suppressed: java.lang.UnsatisfiedLinkError: libmegazord.so: cannot open shared object file: No such file or directory
2021-10-01T02:51:33.6040813Z     at com.sun.jna.Native.open(Native Method)
2021-10-01T02:51:33.6041887Z     at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:204)
2021-10-01T02:51:33.6042743Z     ... 38 more
2021-10-01T02:51:33.8089732Z     Suppressed: java.io.IOException: Native library (linux-x86-64/libmegazord.so) not found in resource path ([file:/home/runner/.gradle/caches/7.0.2/workerMain/gradle-worker.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/unit_test_config_directory/debugUnitTest/out/, file:/home/runner/work/fenix/fenix/app/build/intermediates/javac/debugUnitTest/classes, file:/home/runner/work/fenix/fenix/app/build/tmp/kotlin-classes/debugUnitTest/, file:/home/runner/work/fenix/fenix/app/build/intermediates/java_res/debugUnitTest/out/, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna/5.6.0/330f2244e9030119ab3030fc3fededc86713d9cc/jna-5.6.0.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/app_classes/debug/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ab4ce8d4ea778cadc4a1e5739fe6ae68/transformed/support-test-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-test/1.5.0/403d7911978eb028ddf0c864cc7b22807ecf7588/kotlinx-coroutines-test-1.5.0.jar, file:/home/runner/.gradle/caches/transforms-3/65dc2cf90a17989e538ba22b2ce1c189/transformed/support-test-libstate-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ae38b520f022b18920a7d654f0e8c0d2/transformed/junit-ktx-1.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/141918be8bd74d2b2f8e4a0e90f9b4b9/transformed/junit-1.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8ec5044bfffb9044c3271592ae760fff/transformed/work-testing-2.5.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/robolectric/4.6.1/357f1e5ae699fdaaf02cc8c2152922d4e69dcbbb/robolectric-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-ant-tasks/2.1.3/b09be554228d66d208e5fef5266844aacf443abc/maven-ant-tasks-2.1.3.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk/1.12.0/5c86b2753ab6484e98f93419bf2189927184a575/mockk-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.mozilla.telemetry/glean-native-forUnitTests/41.1.1/bea27df1f95ac9020269427c7f2391e165f6ff4/glean-native-forUnitTests-41.1.1.jar, file:/home/runner/.gradle/caches/transforms-3/3524a701b04aa691f6f04ad22f38635f/transformed/feature-pwa-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d7f02fc2f8561480c836c902f4114a92/transformed/feature-customtabs-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/4c411c3d682790bfdca42cad443a424c/transformed/browser-toolbar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a1ae370a9b59b133e3bfbcc62f9dd3e5/transformed/feature-readerview-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/71d4820a670fb3183727c5692dd9adf4/transformed/browser-menu-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/950dd39f407cc907f61b60df596ce069/transformed/support-migration-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9fdd40e310786d6695a8685a766f6299/transformed/feature-addons-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f7dc662f2f632200debabdae9fc85fa3/transformed/feature-downloads-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/cf703b5de6670cb2173282302c32b85b/transformed/compose-awesomebar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/80edf3ccc96e856cb42cb20c27d10b77/transformed/feature-awesomebar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e6c888715d3127e21402e3265655b5fb/transformed/feature-syncedtabs-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f4a19639e6f4386d212c18c6aba740bc/transformed/concept-awesomebar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5fdf6790a24d5b5b93052124d6443993/transformed/browser-engine-gecko-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d08e729d70f3424ac5fe8a9cab5ca5b0/transformed/browser-tabstray-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7c7c52d6c6fe4b22cdcb1aaa8463f65c/transformed/feature-accounts-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e870e196d5221299ea0adae4505c92e1/transformed/feature-contextmenu-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/90f8c7c3aaf8998aa52d5dd3a03a3733/transformed/feature-intent-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ca207be4f1a919b0b45aa163cbec0acd/transformed/feature-search-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d6757ede70c7439b79b648cec602e9b0/transformed/feature-tab-collections-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c58021f21d1f0a69bdc1d35133b8f73c/transformed/feature-tabs-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/eaffbe479c41fa99a94ed27d1345fda9/transformed/concept-tabstray-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e67d804e60e9b678c89af7d30cbbb99e/transformed/feature-webnotifications-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9aab0070377b7b546a8437611877b82a/transformed/browser-icons-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/70c0a0c2456e84e208d9ae91fb84c3e3/transformed/browser-session-storage-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/21bb6129d1db0fbe5046e130dfda2ea1/transformed/browser-thumbnails-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d1b775ecbc4b8ecf7b89bb6467843c81/transformed/feature-app-links-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/89a172948b7df043e71475f83eb0bbb5/transformed/feature-media-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/46bc8b6ab8f2763b175b4fc5099c7b2e/transformed/feature-logins-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/07f2289746b8be7cb411cd5e188253f1/transformed/feature-prompts-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ae75793e4d226a6fc79ef3fb14511391/transformed/feature-privatemode-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8f08d462e4393c72d3ce55ef7d99ea89/transformed/feature-toolbar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9b31171dbf1682ced5e92f9ef65e4497/transformed/feature-session-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8517e9c5cc3b30977861667c6a2fb675/transformed/feature-findinpage-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c42b24f93f0d2d67fc4ec11d42dc70b0/transformed/feature-sitepermissions-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ee248d6c3401f6f4c8f64db47b1ee979/transformed/feature-recentlyclosed-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/83ee3792166b978dcf723f91dac82ce5/transformed/feature-webcompat-reporter-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d3a0de4ba68a5299a9a17fa737f07cd7/transformed/support-webextensions-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/12701a95bddb43b9a22e2e5b7e38b267/transformed/service-nimbus-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3afc47fd0f7461f2d05aec0402f4e786/transformed/support-locale-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/276c58c73a85eca3005f24ae345c5488/transformed/browser-state-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/bdf5c85dcef6a2eb2afdd8cb89bdf07f/transformed/feature-webauthn-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/45597e18a40acd4620116682c9272e61/transformed/feature-webcompat-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/2708d125a2c7a7f9767d1ac444cccf3f/transformed/concept-engine-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b05d2f7bd3cd494e1d56120ce192da45/transformed/ui-tabcounter-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f8d329ef84c1cd75cc9913ce6e76ada8/transformed/browser-menu2-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/dbc299f7a1a717dcb0acb9b6d3d309bf/transformed/concept-menu-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d7db98ac290c914752d867a6279bec51/transformed/feature-accounts-push-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6aafcb230cffd09d12908ae459b68837/transformed/feature-push-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0b8d24cfaa7a9ad6c5b9b4412b8ce5a7/transformed/lib-push-firebase-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/2075835291c8ddf981608d06f9ad406c/transformed/concept-push-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1795cdb03e6008c80db8807f2fcf1151/transformed/feature-top-sites-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/24c6b330f0354b1c51c38df1964c580d/transformed/browser-storage-sync-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/59296be3932680db4c37b36b14851566/transformed/service-sync-autofill-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6f21ed8c3d76fe1cc9f282608ac5e24b/transformed/service-sync-logins-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0ac4393f4ef96652c71cf7a105f98078/transformed/service-firefox-accounts-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3fb6299bf9ea75263c17e5980c9db657/transformed/concept-sync-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6cfb159886a2465ea0c98cdb92fa654b/transformed/concept-toolbar-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9f31ffbad5eca6d4f179ce5558bb2ba6/transformed/feature-autofill-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/972754c27294d8316808914a95879ad2/transformed/feature-qr-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7aae232278351f364ec74b5136193fe0/transformed/feature-share-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3a696d48b84317c774933bac7165272c/transformed/service-pocket-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ffa8d3870519f2d39844395dbde14453/transformed/service-digitalassetlinks-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/816d30dfd653075f1995a33fbac3296d/transformed/support-sync-telemetry-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e3311a982617d3d68004f78d91ab90ea/transformed/service-glean-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/edc85318e0e8b83b8dcadd3d00a133a1/transformed/service-location-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f99eef156b829698ff66b097e52cf270/transformed/support-images-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/801c0f6394c02f2e9655c2cb8a090751/transformed/lib-publicsuffixlist-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/cf47cd80caa2a9593fdf672ca036ef23/transformed/lib-crash-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/79a6fa67d1e905fa967934a7819474b4/transformed/lib-state-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f421d318749a6c09439355a24764b81b/transformed/browser-errorpages-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1d89a3aee6968ddc940dbc5fd88619c8/transformed/support-ktx-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/93d963a192b227465184c3514e40dee1/transformed/support-rustlog-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6100b5e88e9ef31505d36e378238818e/transformed/ui-autocomplete-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/85be3d85b32564d90653ec046fbf5333/transformed/support-utils-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e1e9a0d41214857ea0819035b1f10ed9/transformed/lib-dataprotect-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f4c953affcc57e288dc75b5d8645c8a0/transformed/support-base-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5c84a50be54943650591fa024a3decd7/transformed/viewbinding-7.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/746683e77e802d040d13e5df144d71a7/transformed/leakcanary-android-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6e55b16e64797af0a5c9b685528ca43f/transformed/concept-base-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0207f98fddae894a47d7562fc011187e/transformed/concept-storage-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-parcelize-runtime/1.5.10/37f48be9c9be4b8c74bf5a5aa17050c18581a370/kotlin-parcelize-runtime-1.5.10.jar, file:/home/runner/.gradle/caches/transforms-3/afd83d015a267446ad132cd48bffd600/transformed/places-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b6bdedaeae68891d4b8cb3696c38ed80/transformed/tabs-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/891a4fbb6e6434cf249634a5729c27ca/transformed/push-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7853a5be97e3cbb3242b451213ec386d/transformed/autofill-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/234375d562c52a9def711ce1566c40a2/transformed/logins-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f2dae8d08e73bc75ff54a05174de27d6/transformed/syncmanager-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a5da0ec537ce1ca7cef58227757e2673/transformed/sync15-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/fed1cac55c013d6371b66a402f3f2287/transformed/fxaclient-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c42b9c4e46b2b96c47285e03a83625cd/transformed/nimbus-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b1c6d02a26890dda1846fb2b57b62fc6/transformed/rustlog-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e9023d4f5f9625f15111f2b30a7b0555/transformed/support-rusthttp-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7048673b6f0d0b54b56e22a321a74eb3/transformed/httpconfig-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/be9ca1dd0727147ded6235d37dc7ed2b/transformed/full-megazord-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f66882ca041139e3dd81971d77936c3e/transformed/native-support-85.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3a0e9254adc3a4547ba6e0fd77500cc1/transformed/browser-domains-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/949c8747cacf87b857dada44e7ffbc7c/transformed/ui-widgets-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/39668193cf2c29dd165bbaffd9a77bdd/transformed/ui-colors-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b0db5f78a2d39e471700ad15a2a53405/transformed/ui-tooling-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/86b0a71d2a3c5b65881060a4c679c6f7/transformed/material-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9a30393d43acb816b35de980cc10635e/transformed/material-ripple-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c2ad0a3c7c4acd7f3628c900c45d7676/transformed/foundation-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c7169241fff195428b4383df7af904c1/transformed/ui-tooling-data-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/13bb243931ba55ac7025ec7de3a66c30/transformed/activity-compose-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d0755e08029f00c1f5834073deac4392/transformed/animation-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/33635c92682efcb1b0d71e62c35e094f/transformed/foundation-layout-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a5c310641e152f921d3c451efee01543/transformed/animation-core-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3b388578844252cc1593d616be3f274d/transformed/material-icons-core-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8097edcfe0a0bb39fb8088f357f86400/transformed/ui-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5039c9b5c9dd162b9f5b5e5fe94bd64f/transformed/glean-41.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/43d6028aa6312f58718c416613958988/transformed/work-runtime-ktx-2.5.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/87a8f2b378bc769a71cdb1f49fa3d635/transformed/room-ktx-2.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8c3fb3f5cc7b47eb058904f6695b1c65/transformed/ui-text-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9b14c60d083285ce8ff27b050d779ed8/transformed/runtime-saveable-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/4072c15d001370683286c271341865ce/transformed/ui-graphics-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/581c1289c528e1f9066d915a1c00102f/transformed/ui-unit-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/299b2cfbe32cac605c184e893423dcdc/transformed/ui-geometry-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ece49830db201e3dd84b1810bb83290c/transformed/ui-tooling-preview-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8c2a1b3ec2677ceddf18fc2a58ec2e5d/transformed/runtime-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3cdf9e5e246f63158bcf278d7e0c3fd8/transformed/preference-ktx-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/98f1441f5369b3098c93c4f555cd8d9d/transformed/navigation-fragment-ktx-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/83709bec1b283d4588b9f6d9fbe734ef/transformed/fragment-ktx-1.3.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/982a087c2f1e249ce4c80742e15fae49/transformed/navigation-runtime-ktx-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c45676d0e26dd3b6d78ee2a1c1da7169/transformed/activity-ktx-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0a00ee34c1873160352b6e160a35266a/transformed/lifecycle-viewmodel-ktx-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f4ea583d619b44e65e6b68711eb164f9/transformed/lifecycle-runtime-ktx-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-android/1.5.0/c1df91b076cf9a1af564bb7edd41c49683169e25/kotlinx-coroutines-android-1.5.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-debug/1.5.0/1406a87f78bfe3173af2b00d2710b053fb7ea77f/kotlinx-coroutines-debug-1.5.0.jar, file:/home/runner/.gradle/caches/transforms-3/9cdbda04e3169db96e2138814d95b632/transformed/lifecycle-livedata-ktx-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core-jvm/1.5.0/d8cebccdcddd029022aa8646a5a953ff88b13ac8/kotlinx-coroutines-core-jvm-1.5.0.jar, file:/home/runner/.gradle/caches/transforms-3/8d171355db3f1a761d7fb1ff451904db/transformed/geckoview-nightly-omni-94.0.20210930095024-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.5.20/636c5653641cd956de9aee5792155b07ea49825e/kotlin-stdlib-jdk8-1.5.20.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.5.20/218b60e1d446d1e0a18bc7aa8663634b136fbcc5/kotlin-stdlib-jdk7-1.5.20.jar, file:/home/runner/.gradle/caches/transforms-3/e3d23334357ca594c813df22bdeb708e/transformed/constraintlayout-2.0.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ed2f5352e9f86436513b3a7603b5a532/transformed/biometric-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b43d916e40e98cce64c5e0f3e6398ac0/transformed/navigation-ui-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9a396a2fcb9b0e04039edac7a6af15c9/transformed/material-1.2.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d9f9770bab99e9b087f2204dc7af36ad/transformed/preference-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7df94cb15a1b31b3cb97d29d1131cdc1/transformed/appcompat-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/36f4a612e6463f4ad2d8b1062f539028/transformed/legacy-support-v4-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f7e1dec922f964372b1b804f74dab8d8/transformed/legacy-support-core-ui-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3c61b90a234c04039b2551b8e2e69db5/transformed/coordinatorlayout-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.sentry/sentry-android/1.7.10/f70c4e3611686df8f5c1eda094313257418b135a/sentry-android-1.7.10.jar, file:/home/runner/.gradle/caches/transforms-3/3fee1c223c37de45d68994686c8afba7/transformed/ui-icons-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f154bd74cb54ca2cee9d8e51806361dc/transformed/paging-runtime-ktx-2.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/42dd6153ce6e7f70bd4db44c68d9f9dd/transformed/paging-runtime-2.1.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a06a4eebee0624c5dfa521af5baa5bea/transformed/viewpager2-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/38b5937f231c9174cca1d2285e06b660/transformed/recyclerview-1.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common-java8/2.3.0/cd3478503da69b1a7e0319bd2d1389943db9b364/lifecycle-common-java8-2.3.0.jar, file:/home/runner/.gradle/caches/transforms-3/f8364a66f80d9061959251ab7caa03f6/transformed/lifecycle-extensions-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1df0f70a1af1405246dda0d95623ba94/transformed/lifecycle-process-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/0ab83552e28d378cb3ce60e140dd4fb6/transformed/navigation-common-ktx-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/26a223b8244bc65be1e01eb71d2be81e/transformed/core-ktx-1.3.2-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5edb318c6ecbdc1ba00b06ec8ff457e3/transformed/transition-1.4.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/63f7fe2142c454a7f67036cc6f5385b9/transformed/navigation-fragment-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d3f012e79571635d0e1f5c80395c0957/transformed/play-services-ads-identifier-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b95175af4adc71285a193ea421205429/transformed/firebase-messaging-20.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a1bc4a9e962f0d7b4bd392fa4a104829/transformed/firebase-iid-20.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3933de80dce0189a935fa61ada021f95/transformed/play-services-cloud-messaging-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/5eb2072d6b3cea045411e65e98838c2d/transformed/play-services-stats-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6c9c3d503e76924b073d9712babb934b/transformed/firebase-datatransport-17.0.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ec37a2b5c487f411271c65243e695109/transformed/firebase-installations-16.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1f9f2db55779f3ebdfa681d0ec613712/transformed/firebase-common-19.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7d2fe8e311b405d00adf4423565f214d/transformed/firebase-installations-interop-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/daceeac7767e7efa7837c97519184087/transformed/firebase-iid-interop-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b471ced3bb768a7adc33a48e0c6b2378/transformed/play-services-fido-18.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a4ba4c8bafc8b4dd2b266ee4ed5619ae/transformed/play-services-base-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/30f0bbc6df9760c9d0316f1b9b83f4ef/transformed/play-services-tasks-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a176e95c95a09e897c5d032363de48ee/transformed/firebase-measurement-connector-18.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/30c5b9dae0efda29c2c74a55c1bead39/transformed/play-services-basement-17.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/09e9cece8e01c5ba1cf494eecb987981/transformed/fragment-1.3.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/fed3ed57d186ea15a3dc8408d950b16b/transformed/appcompat-resources-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/150fd957b52af41a0ebdc23195c8e963/transformed/browser-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/79aa8f50b8d555715ad2234e821b2be5/transformed/media-1.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f173018db664a1a1a5dc45bda939f61f/transformed/swiperefreshlayout-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/f7d8923619223099f0b46d474f80a6eb/transformed/autofill-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a7889331eccbf5f0c19a288276649871/transformed/palette-ktx-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3380adb75e708561dd8ad877e5d05026/transformed/palette-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/903326348dd0f99acb1dc89f0cd866e9/transformed/legacy-support-core-utils-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/3914dfdddf4ce77ae6338a9dee804ef3/transformed/drawerlayout-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1648b3c23b7cacbcb00db10e68b94ee7/transformed/viewpager-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7b7e0edaa9a4f8c4909fc7e527a3a6b4/transformed/slidingpanelayout-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8bc778553044aac12c4199eb654a10ac/transformed/customview-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9a1abe271afc7bf6929561d3e145d079/transformed/work-runtime-2.5.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/959e9da2e52ab63e586bab5a9f3b8811/transformed/vectordrawable-animated-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/07e596b24e5d431a3c6ab887f48a85a1/transformed/vectordrawable-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/580649d2970d5ae51beb4eee062a6a16/transformed/loader-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/73fd4a768136cf8de32a3fa3e7c3189d/transformed/asynclayoutinflater-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/42e9b060475236c104f1e93451a1416c/transformed/navigation-runtime-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/71efa7d87055b7ae23827ccf51edbda8/transformed/navigation-common-2.3.3-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9540726bd958458e5239bb399a4e78ce/transformed/activity-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1d026474ab1e44e37dd4e792bb62843c/transformed/core-1.5.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.adjust.sdk/adjust-android/4.18.3/af89b4dc961127355d7072b1423653b64a983154/adjust-android-4.18.3.jar, file:/home/runner/.gradle/caches/transforms-3/4fbc51a78232e43b61c8fb3e9a4ddb4b/transformed/installreferrer-1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7ae05f4498bbba343754926d9d2c3866/transformed/core-1.8.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c27602bf84caf3d360f125296d933ad2/transformed/falcon-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-agent-jvm/1.12.0/96421a1df31560296f7879a15953c5ef8637b049/mockk-agent-jvm-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-dsl-jvm/1.12.0/eb517db1a4dea71a3bdce5065b3b280b559ff958/mockk-dsl-jvm-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.5.10/d6c70f3c0df2457ea0095c61c1fc1188017dc3bc/kotlin-reflect-1.5.10.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-common/1.12.0/3d3fd6c65c1f6aa66210b8b5f581097d6a0d5e0e/mockk-common-1.12.0.jar, file:/home/runner/.gradle/caches/transforms-3/0266a6c4e0fe5f6738279cdbc40dc702/transformed/leakcanary-android-core-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/a71ad76907b1a39fcc369e0f701e1eda/transformed/concept-fetch-94.0.20210930211433-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/c7470aabd6845643e880878d8d6fa890/transformed/ui-util-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.paging/paging-common-ktx/2.1.2/3acd13bd21e696857fe39989070c1018ec79166a/paging-common-ktx-2.1.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.collection/collection-ktx/1.1.0/f807b2f366f7b75142a67d2f3c10031065b5168/collection-ktx-1.1.0.jar, file:/home/runner/.gradle/caches/transforms-3/47a36326d973acf0fd654fb55ef9d759/transformed/lifecycle-livedata-core-ktx-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/529627f676985c0f763ccdc3b422c82b/transformed/savedstate-ktx-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-agent-common/1.12.0/bbca73456f85b41b03b6d11414ebbf19360848a4/mockk-agent-common-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-agent-api/1.12.0/415fd3961ba729257ddfe6639be6124869335ca8/mockk-agent-api-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.mockk/mockk-dsl/1.12.0/abe7c99dd6b8a2b8d2e10c2b103d36775c10b8a9/mockk-dsl-1.12.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-android/2.4/d07c172a763fbfd0e9450f1ba1480ce4c633fdd3/shark-android-2.4.jar, file:/home/runner/.gradle/caches/transforms-3/0da74ab4a834cbab501f7d01b43b45fb/transformed/leakcanary-object-watcher-android-androidx-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/ab70d29508b64e9aa1c12c0988bf2892/transformed/leakcanary-object-watcher-android-support-fragments-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/d4fa3074a9779db7fab01f7bdff2f77e/transformed/leakcanary-object-watcher-android-2.4-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1957cccd6dc45a1124d02fe4fcc058be/transformed/plumber-android-2.4-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-android-extensions-runtime/1.5.20/9a565217f3ba7b873ffe0617d0c681929d82d893/kotlin-android-extensions-runtime-1.5.20.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark/2.4/5aa97ac9e71a6127388e13a8ea070586f41f4014/shark-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/leakcanary-object-watcher/2.4/a40f33d9b1c5218e6da063dc996c89d0dac122d1/leakcanary-object-watcher-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-graph/2.4/89557baa9c59c4bf0e667905942966f3513d5cf6/shark-graph-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-hprof/2.4/367ac7b4f37a64747fcf5327258cd5f82496bf89/shark-hprof-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.leakcanary/shark-log/2.4/92aa5a581103961c3b56b31e27bb349def942498/shark-log-2.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.squareup.okio/okio/2.2.2/36f483536153f15339a8b48d508e22be7c9c531a/okio-2.2.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.5.20/9de35cc611bcecec8edce1d56d8e659953806751/kotlin-stdlib-1.5.20.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.mockito/mockito-core/3.11.2/bb97012477172d7c04ff2193e31fcb38dd171879/mockito-core-3.11.2.jar, file:/home/runner/.gradle/caches/transforms-3/af2185c3b044ab6839bf78bd9be3b301/transformed/core-ktx-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-project/2.2.1/8239e98c16f641d55a4ad0e0bab0aee3aff8933f/maven-project-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-artifact-manager/2.2.1/ec355b913c34d37080810f98e3f51abecbe1572b/maven-artifact-manager-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-error-diagnostics/2.2.1/e81bb342d7d172f23d108dc8fa979a1facdcde8e/maven-error-diagnostics-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-settings/2.2.1/2236ffe71fa5f78ce42b0f5fc22c54ed45f14294/maven-settings-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-profile/2.2.1/3950071587027e5086e9c395574a60650c432738/maven-profile-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-plugin-registry/2.2.1/72a24b7775649af78f3986b5aa7eb354b9674cfd/maven-plugin-registry-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.plexus/plexus-container-default/1.0-alpha-9-stable-1/94aea3010e250a334d9dab7f591114cd6c767458/plexus-container-default-1.0-alpha-9-stable-1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.12/2973d150c0dc1fefe998f834810d68f278ea58ec/junit-4.12.jar, file:/home/runner/.gradle/caches/transforms-3/bc2a85f52e4b03241a838d0978a4fa95/transformed/core-1.3.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/shadows-framework/4.6.1/f434e4745c28ec2e75ca58ab81fa0678c4c6be74/shadows-framework-4.6.1.jar, file:/home/runner/.gradle/caches/transforms-3/518de95027b90bd01ce639d2928ddf70/transformed/monitor-1.4.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1296e7608087561c216bd8455fd8822f/transformed/cursoradapter-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/cb2a9375ad8202f00eae50b1c0d91eac/transformed/lifecycle-viewmodel-savedstate-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1e6ebf045bc5881e77c8afc03e30d96c/transformed/savedstate-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/8ef45442972f965c609c68b3508c82e4/transformed/lifecycle-service-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/db10c13ff81eb2de59589a435e97b66c/transformed/lifecycle-runtime-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/45dd3aeb96a1f09d97c80c5a177d00c6/transformed/lifecycle-viewmodel-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/386fd8e4f40b82b438ba4fc30baee74a/transformed/cardview-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/6e72a666f90f1cc366b5be76a10cdd89/transformed/localbroadcastmanager-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/2559d1d917a3c0e65caf6c14e8904afc/transformed/profileinstaller-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/e65e725e9ac47184a44ff7ba5c712eb1/transformed/versionedparcelable-1.1.1-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.collection/collection/1.1.0/1f27220b47669781457de0d600849a5de0e89909/collection-1.1.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.paging/paging-common/2.1.2/e8cdd44767173c4af5164732c32741c89afa684d/paging-common-2.1.2.jar, file:/home/runner/.gradle/caches/transforms-3/0c3052771365bfed11824b32a5339905/transformed/startup-runtime-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7746c89b92c26c5a20c44f13a8dffef7/transformed/tracing-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/b57871dbd59e51561b9af525796a385f/transformed/lifecycle-livedata-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/74544aac2b3ab49ec5c1b881137ce2ec/transformed/lifecycle-livedata-core-2.3.1-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/499d422fc2e1ff5ffc5ea21b644320ef/transformed/room-runtime-2.3.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/73be4a49f8025a97bbbd42e8ce9eb8c6/transformed/core-runtime-2.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.lifecycle/lifecycle-common/2.3.1/fc466261d52f4433863642fb40d12441ae274a98/lifecycle-common-2.3.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.arch.core/core-common/2.1.0/b3152fc64428c9354344bd89848ecddc09b6f07e/core-common-2.1.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.concurrent/concurrent-futures/1.0.0/c1e77e3ee6f4643b77496a1ddf7a2eef1aefdaa1/concurrent-futures-1.0.0.jar, file:/home/runner/.gradle/caches/transforms-3/efe3f3ee1dc30a1c9cd84271b15e271a/transformed/interpolator-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.room/room-common/2.3.0/11e5b5194cb53b2c5ebfa57710a1a74c67cb3335/room-common-2.3.0.jar, file:/home/runner/.gradle/caches/transforms-3/34e6d0d4aa93b1685d645dd28b22bb6f/transformed/transport-backend-cct-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/7c5000aa5c1863c933bec5e4c6806c98/transformed/transport-runtime-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/18e628fc0f7b6044b72919839e900ec4/transformed/firebase-components-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/19b6e592012085f30ca322a3e3a5815b/transformed/firebase-encoders-json-16.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/273c27a557d721002b28ffb800682fcb/transformed/documentfile-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/bbd96c7293056e8b5ac41a4087c28f7f/transformed/print-1.0.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/9bc90ff6346a510ed11e151ba9ad693b/transformed/sqlite-framework-2.1.0-runtime.jar, file:/home/runner/.gradle/caches/transforms-3/1dd29afe92de61dc44220398bcc6f41f/transformed/sqlite-2.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation/1.2.0/57136ff68ee784c6e19db34ed4a175338fadfde1/annotation-1.2.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/junit/4.6.1/4332f14f797763028dbcec462b7956a507320690/junit-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/resources/4.6.1/444de1a4b500db4531bc355c395a8cf4655328a4/resources-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/sandbox/4.6.1/6f07458cd93eb90a68679d9f64b9fc7057890f4b/sandbox-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/utils-reflector/4.6.1/409d6c0557f8f2a59782483a5813ac50b75bde54/utils-reflector-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/plugins-maven-dependency-resolver/4.6.1/b16422d286873448f3deb8737657c2aff876b546/plugins-maven-dependency-resolver-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.6.1/2df9b53fac09615e6bc0c77ee36b8483a05ee825/shadowapi-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/utils/4.6.1/d14910e168c8a931fde766e64b49fc5d3f1f7976/utils-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/pluginapi/4.6.1/95d8b0a2afb69c8de1e56abc02728e09cf60bf25/pluginapi-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.robolectric/annotations/4.6.1/246a0f0bd16253b7d51e0bf2ef3646bf38acdbc5/annotations-4.6.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.dagger/dagger/2.24/97f35169621dd6a30d5958e8326e30f76ebe2e62/dagger-2.24.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/javax.inject/javax.inject/1/6975da39a7040257bd51d21a231b76c915872d38/javax.inject-1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/javax.annotation/javax.annotation-api/1.3.2/934c04d3cfef185a8008e7bf34331b79730a9d43/javax.annotation-api-1.3.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.68/46a080368d38b428d237a59458f9bc915222894d/bcprov-jdk15on-1.68.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.ant/ant/1.8.0/7b456ca6b93900f96e58cc8371f03d90a9c1c8d1/ant-1.8.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/classworlds/classworlds/1.1-alpha-2/5adf2e681c57d7f48038b602f3ca2254ee82d47/classworlds-1.1-alpha-2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-artifact/2.2.1/23600f790d4dab2cb965419eaa982e3e84c428f8/maven-artifact-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-model/2.2.1/c0a1c17436ec3ff5a56207c031d82277b4250a29/maven-model-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-file/1.0-beta-6/6c53633505460caf49d2660de1e24744d915afb9/wagon-file-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-http-lightweight/1.0-beta-6/b3815078570c3b1f0667e123d59717c6b726c6c2/wagon-http-lightweight-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-http-shared/1.0-beta-6/ccd70d7e0d8c085e648a83f072da06ca9a53be94/wagon-http-shared-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven.wagon/wagon-provider-api/1.0-beta-6/3f952e0282ae77ae59851d96bb18015e520b6208/wagon-provider-api-1.0-beta-6.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.maven/maven-repository-metadata/2.2.1/98f0c07fcf1eeb213bef8d9316a9935184084b06/maven-repository-metadata-2.2.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.plexus/plexus-utils/1.5.15/c689598ce1eb94c304817877ed15911099972526/plexus-utils-1.5.15.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.plexus/plexus-interpolation/1.11/ad9dddff6043194904ad1d2c00ff1d003c3915f7/plexus-interpolation-1.11.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna-platform/5.5.0/af38e7c4d0fc73c23ecd785443705bfdee5b90bf/jna-platform-5.5.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.java.dev.jna/jna/5.8.0/3551d8d827e54858214107541d3aff9c615cb615/jna-5.8.0.jar, file:/home/runner/.gradle/caches/transforms-3/5d30a58e1dc4d81c0ebc75d9c1c4bcd9/transformed/jna-5.8.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/androidx.constraintlayout/constraintlayout-solver/2.0.4/1f001d7db280a89a6c26b26a66eb064bb6d5efeb/constraintlayout-solver-2.0.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/io.sentry/sentry/1.7.10/2a2fd502a52f9f067b6f34f3111aff7803b66d25/sentry-1.7.10.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.jakewharton/disklrucache/2.0.2/93b399470e72c0bc8cb52379943b63c9aa586cd8/disklrucache-2.0.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.zxing/core/3.3.0/73c49077166faa4c3c0059c5f583d1d7bd1475fe/core-3.3.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.5.20/3d79dbd48bf605f4aac1e7028981a1953e245cbb/kotlin-stdlib-common-1.5.20.jar, file:/home/runner/.gradle/caches/transforms-3/81c8199effec3759afc59c9924126454/transformed/annotation-experimental-1.1.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.jetbrains/annotations/13.0/919f0dfe192fb4e063e7dacadee7f8bb9a2672a9/annotations-13.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.11.5/3145358bfab3d86aebac5f31c7703505ebb108c1/byte-buddy-1.11.5.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy-agent/1.11.5/ec760e507143fb9696595150a03573b066f46887/byte-buddy-agent-1.11.5.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.objenesis/objenesis/3.2/7fadf57620c8b8abdf7519533e5527367cb51f09/objenesis-3.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/27.0.1-jre/bd41a290787b5301e63929676d792c507bbc00ae/guava-27.0.1-jre.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-commons/9.0/5a34a3a9ac44f362f35d1b27932380b0031a3334/asm-commons-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-util/9.0/7c059a94ab5eed3347bf954e27fab58e52968848/asm-util-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-analysis/9.0/4630afefbb43939c739445dde0af1a5729a0fb4e/asm-analysis-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm-tree/9.0/9df939f25c556b0c7efe00701d47e77a49837f24/asm-tree-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.ow2.asm/asm/9.0/af582ff60bc567c42d931500c3fdc20e0141ddf9/asm-9.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.almworks.sqlite4java/sqlite4java/1.0.392/d6234e08ff4e1607ff5321da2579571f05ff778d/sqlite4java-1.0.392.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.ibm.icu/icu4j/53.1/786d9055d4ca8c1aab4a7d4ac8283f973fd7e41f/icu4j-53.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.auto.value/auto-value-annotations/1.7.4/eff48ed53995db2dadf0456426cc1f8700136f86/auto-value-annotations-1.7.4.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.ant/ant-launcher/1.8.0/8b53ba16fa62fb1034da8f1de200ddc407c8381/ant-launcher-1.8.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/backport-util-concurrent/backport-util-concurrent/3.1/682f7ac17fed79e92f8e87d8455192b63376347b/backport-util-concurrent-3.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.24/1da441001b817bc03a53e6393f2014e59e1438a/snakeyaml-1.24-android.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.24/3f6b4bd4f8dbe8d4bea06d107a3826469b85c3e9/slf4j-api-1.7.24.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.8.7/8b46f39c78476fb848c81a49fa807a9e9506dddd/jackson-core-2.8.7.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava/b421526c5f297295adef1c886e5246c39d4ac629/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.protobuf/protobuf-javalite/3.11.4/e3b3f442b01f8fa9bb54893abc44aa352c9b1afd/protobuf-javalite-3.11.4.jar, file:/home/runner/.gradle/caches/transforms-3/72bfcb85c4f3dc5da9f607c6de4f79da/transformed/transport-api-2.2.0-runtime.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.guava/failureaccess/1.0.1/1dcf1de382a0bf95a3d8b0849546c88bac1292c9/failureaccess-1.0.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/3.0.2/25ea2e8b0c338a877313bd4672d3fe056ea78f0d/jsr305-3.0.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.checkerframework/checker-qual/2.5.2/cea74543d5904a30861a61b4643a5f2bb372efc4/checker-qual-2.5.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.errorprone/error_prone_annotations/2.2.0/88e3c593e9b3586e1c6177f89267da6fc6986f0c/error_prone_annotations-2.2.0.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/com.google.j2objc/j2objc-annotations/1.1/ed28ded51a8b1c6b112568def5f4b455e6809019/j2objc-annotations-1.1.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.codehaus.mojo/animal-sniffer-annotations/1.17/f97ce6decaea32b36101e37979f8b647f00681fb/animal-sniffer-annotations-1.17.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/nekohtml/xercesMinimal/1.9.6.2/d1c5e063683a0e6f77cd5f051a9d4af48346fa6/xercesMinimal-1.9.6.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/nekohtml/nekohtml/1.9.6.2/2d960be7b62ae6622dbbbe49ab4ffdc609f85c80/nekohtml-1.9.6.2.jar, file:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/java_res/debug/out, file:/home/runner/.gradle/caches/transforms-3/ba191990df0f6ee7059779aaa5f3ab6f/transformed/support-test-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7ca3872a669cf94ba4aabdfe019cb0bd/transformed/support-test-libstate-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/42fa9d0501667c17c5b01ba40d48e4a7/transformed/junit-ktx-1.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/db22c41db31da5b8807029623ef7bd29/transformed/junit-1.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/380f889f9f31dd996ce5098eff96134a/transformed/work-testing-2.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7dbceeb74d0b10830e9cf8aae06f05d5/transformed/feature-pwa-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/093d8a56d66dd9ec95459678bb72aa7f/transformed/feature-customtabs-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1ff4447287b325c66dfec661addea073/transformed/browser-toolbar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2326b42998c8dbebf1865d32533ef6fb/transformed/feature-readerview-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/759ba40053b4495804d4b30507a32800/transformed/browser-menu-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c140eff10d6deac8dad236130a4e1dcb/transformed/support-migration-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/43580059c4d6da694f065767b65f70ad/transformed/feature-addons-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0a49382d4e4bca7d07c7c86dda97af92/transformed/feature-downloads-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/15c11f3bf762162978a5237479999ccd/transformed/compose-awesomebar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/77fc7440ccc4f4e6cf81285b336c8330/transformed/feature-awesomebar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2bc01451e492f9aa47b9ba562e3aad6f/transformed/feature-syncedtabs-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c634479160fd1f87b909e269841af291/transformed/concept-awesomebar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/27013ace9d93b5c35f3b9ba27d781b47/transformed/browser-engine-gecko-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0825e5ded0cd3cf326e5df75be59d5a2/transformed/browser-tabstray-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/14da3df9b97111880cb76a221477a548/transformed/feature-accounts-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20382eb9488703791ef9d36fe646529c/transformed/feature-contextmenu-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/987368d91f4b57017323d25e8fd00373/transformed/feature-intent-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0cea274b6d8031572389a38b8f5ed1f6/transformed/feature-search-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/42c89d3ffcc53783754e7a771f9bc1f4/transformed/feature-tab-collections-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e2ccd78d5e14a477f4f02a1973599a4b/transformed/feature-tabs-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/15f71e649444277f48b59e9159ada092/transformed/concept-tabstray-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7d8fab755d1a1f239c5a84d1eaa34095/transformed/feature-webnotifications-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0a9a33ec51107b80ef499fd0140c2078/transformed/browser-icons-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/54071b7a3eb211f882d05453e225bdb5/transformed/browser-session-storage-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e49f6c17701941bfbc4f376be7750138/transformed/browser-thumbnails-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/db74f0b6c93e6559a184005e94a655a8/transformed/feature-app-links-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2af7b54513f20009493e5261f4e68753/transformed/feature-media-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d4bfa18126161bf331d2f37552fe3412/transformed/feature-logins-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4da75c1f9005cdc0ae9481d1fdb3dd11/transformed/feature-prompts-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bea34efdad5f410ba63d5cb6249b7ab9/transformed/feature-privatemode-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b94de645380f13afee5e46b7247bb199/transformed/feature-toolbar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5cdda9636b56247af2be75a55c1c15cd/transformed/feature-session-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f8c9f8a8ca6e555c0d58505cb16a95c3/transformed/feature-findinpage-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/94ddcc084014b87a316f061ecfb0e672/transformed/feature-sitepermissions-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/32cf17543c6631dcd9aaf06b802046a6/transformed/feature-recentlyclosed-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/668ac8f2d4281349215e37f9f635251f/transformed/feature-webcompat-reporter-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/69aeb71e52c2cfb1c5bfb0030f6a9335/transformed/support-webextensions-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fe0b54d3ddba6bba80833cda32a3298d/transformed/service-nimbus-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5a79bbfb1ee7bc8016ceadf7d2c3768c/transformed/support-locale-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/69bd7e65580e13ce3a38c1315e3cfb79/transformed/browser-state-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0fd4593ef384c47dd4527056629e3fe1/transformed/feature-webauthn-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/01e3ed1f6937932128eea00820cdcd1a/transformed/feature-webcompat-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/71ca13a14f81740e453cb70bafde7e8c/transformed/concept-engine-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/36f77a54927d1b9c5ceeb2d6570a2685/transformed/ui-tabcounter-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d75e8fd2555d845de178b9853bea1628/transformed/browser-menu2-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f1391e46b82714b92cdf4f6931decf5e/transformed/concept-menu-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/39d2fd2db1d611d20dcec7eb3bd47dd4/transformed/feature-accounts-push-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d8754f401e2dc6696412dbec224722dc/transformed/feature-push-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1a86558ca2761a9ab8dad55b9e7cc858/transformed/lib-push-firebase-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e74ffae6b8af80fa56d22340e090196e/transformed/concept-push-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5c32d4574c1c0fabf62c7060a4ae950e/transformed/feature-top-sites-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e1c58271a6850edb5cba7b3890712452/transformed/browser-storage-sync-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/71af42e4e8705719794bfd9db35800d0/transformed/service-sync-autofill-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/21a2593e8602564c2d944781d5ab2f57/transformed/service-sync-logins-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1d66762d16d191a821953258d29c6f38/transformed/service-firefox-accounts-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/19a44b067d8eb375611f229bd17bb341/transformed/concept-sync-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/83835604b0c17bbba06b4b9c982564c4/transformed/concept-toolbar-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/01a6a132e42a947026a5d3d07931e31e/transformed/feature-autofill-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/13c9e3533dd67fa4ab4a136c692353e8/transformed/feature-qr-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9db59d32f5f44a7e261cbb34f70d7459/transformed/feature-share-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20f3cea81f89cdd505ef1fbbb9443fc9/transformed/service-pocket-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b3b39bf157a71d54061591dae020bc07/transformed/service-digitalassetlinks-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d28be1f1fa24299b84b6bfe0b48fe608/transformed/support-sync-telemetry-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d4e65b6eacbbffa4b09a7ced543af578/transformed/service-glean-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fdbcb818d21cfadb5101cefd38d83d9f/transformed/service-location-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/366e806dd7a3416b1482b329f1fd4bb3/transformed/support-images-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/70e1aac7f4cd159d14cccfc3d2f551a9/transformed/lib-publicsuffixlist-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/de6f972524db9978bf0af70f1e2cff1b/transformed/lib-crash-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5aa67e3c78bfec285ab46fb82bbcc0ed/transformed/lib-state-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bfc9517a6467b29b8b94b88b3bcc6364/transformed/browser-errorpages-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b0a48eaeee227438fdd980d4afee15a2/transformed/support-ktx-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/07c21d906d6b2dffa4eaf539286c024e/transformed/support-rustlog-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c96eb0cef4ce3476cfb785475f2215f8/transformed/ui-autocomplete-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/05ed3cdb893628e006c57b4b9deb6140/transformed/support-utils-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ed7c22196613f9818519e173ce026472/transformed/lib-dataprotect-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d1cb066697fb607634f1214d875aaef5/transformed/support-base-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b82cb7a9ee82241031d41d1a85839609/transformed/viewbinding-7.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2aef12d5a7eae920c520d689d485f900/transformed/leakcanary-android-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ed063d68580f8c41f2d0a5ddf12c1312/transformed/concept-base-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0c3a98692d77145bbeba0b4a138b9f5a/transformed/concept-storage-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/8a7a87ab86002820e06d01f01ccc7bea/transformed/places-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ab724b887f2b12015f02275ad2900ad0/transformed/tabs-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4811137968708d328838823dcc4cbf09/transformed/push-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ece57c3a8f016c30da4ed75f1e187c15/transformed/autofill-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ed1372b3267067e9f943b5f6e08fcea6/transformed/logins-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20d17ce1c1839670a8abe744c163d1bf/transformed/syncmanager-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bdc5d20032f3a2bc5a6cb233879b04b4/transformed/sync15-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/cd5f27c89abc06789b2200d682aa45e3/transformed/fxaclient-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5371440af6a0fb710dc568c7adfbe9c3/transformed/nimbus-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b9f83d0b8df553d24d15d36f6e35b8d3/transformed/rustlog-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4f09cc39fa145db62fc7c48921763456/transformed/support-rusthttp-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/60f433cddd20f846621020fd82e15275/transformed/httpconfig-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d89c11a086e1c372739b5237e6041eb4/transformed/full-megazord-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/297451f1989f239923049c16a3e4670d/transformed/native-support-85.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e531b0bc3a99065ac3c8fdd51403d1a3/transformed/browser-domains-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/baf9825fc08199deae594085033f277d/transformed/ui-widgets-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/811277e4ea26343b3293d26612b7482c/transformed/ui-colors-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/8f38f873c37b9d5c42d6293ae7082cdb/transformed/ui-tooling-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e1d9434246f02fda28323fb79c693000/transformed/material-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/07423513a63e491c6d9ae9ce9ee6840e/transformed/material-ripple-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ccc2cb572135c1abf4b88c2598a7e618/transformed/foundation-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e58f01de349caecfdeb7798ec4dc83c5/transformed/ui-tooling-data-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a01a4f4f554bedf668b35814fa96845a/transformed/activity-compose-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/05d5a3f014fe04a0fe51a3de9b941fae/transformed/animation-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d3c3a817f279394983587c9d9da69e92/transformed/foundation-layout-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e1c7ce8fad4a5fdf7a87f2b6eb533071/transformed/animation-core-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3e48c0cf5911f82561e38f77bb6afa4d/transformed/material-icons-core-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/abe560afde1d89f3f66600291f758bc5/transformed/ui-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e47b09aeaca63b282cc110eba5ebb8d9/transformed/glean-41.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e993a2e5b27084fc183d8ade17c4d3c6/transformed/work-runtime-ktx-2.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4dc3ffb354b95ab4f58ac3dca3dc06ae/transformed/room-ktx-2.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/73f6caa643aa0dfe24fbd443e107dab3/transformed/ui-text-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/853de0dda20803883bd992c4674ef8cd/transformed/runtime-saveable-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/91f27f0fa2633bbb79ef6df7e3561e1c/transformed/ui-graphics-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1ccfb84cfe130eb88f52c2b7e088cb10/transformed/ui-unit-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/30d4f7e60955de59b3f27ebbf2e3ec06/transformed/ui-geometry-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6238f88043642cc0c4acd7b8a7ec04c2/transformed/ui-tooling-preview-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4d3b9344c0e1f2f1418b25fc2ae12865/transformed/runtime-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/603606fccb1c344df8ee7c76ffeeb16b/transformed/preference-ktx-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a72ed687c79c4c54fe0d9b299d21db45/transformed/navigation-fragment-ktx-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a87b0a3a8a87252e15cd837f7f0185d0/transformed/fragment-ktx-1.3.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/afa77158e0883c7b9e89899a1a913e87/transformed/navigation-runtime-ktx-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1bfc24c8c978434d0b885d2467faf54e/transformed/activity-ktx-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6e40a7e0dd995dd5674680bbcd874d26/transformed/lifecycle-viewmodel-ktx-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4a10a829855e083dbcdef8f1ce95ed1e/transformed/lifecycle-runtime-ktx-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/bf223090195bea59f6ea2b85e3f4345b/transformed/lifecycle-livedata-ktx-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9dbc490c6adfaf3d68a8ebfa41b8eadf/transformed/geckoview-nightly-omni-94.0.20210930095024/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/115366cfb9f40c21fe292087bc3f790d/transformed/constraintlayout-2.0.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7b4bd1add714033a504aaf9cb031fb68/transformed/biometric-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6ba5a61902f74993792f2cb8f7f4e098/transformed/navigation-ui-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/26d37146b3c97d6a500d5530d371e156/transformed/material-1.2.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a4bd908a89b20eff8a6571dd9a83266d/transformed/preference-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9422d6bb91cfaafbfb5bc59f84755587/transformed/appcompat-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/412946f166f3f67f1feaf3dc46fa7529/transformed/legacy-support-v4-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/1f3cb2469a8de97c51aed231687f2615/transformed/legacy-support-core-ui-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/67e7e04105ee2d03cfd4da1706e1da26/transformed/coordinatorlayout-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/91ca8fece7e84b783f009b70aa6cd502/transformed/ui-icons-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e8e16c27d463e3c84075ae9ebef66578/transformed/paging-runtime-ktx-2.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/dece0ebfae7173b3c6d7e875e3a9bab1/transformed/paging-runtime-2.1.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fe77c872280f791e6047f22693fc4b91/transformed/viewpager2-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/645d044dba9de8a7b2f311f022cf9f83/transformed/recyclerview-1.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/44cdd0edc77e18fbf201ef53523f1bf5/transformed/lifecycle-extensions-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/53d1cd5baa65a8a895665a52fdfa23bb/transformed/lifecycle-process-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f72319d3153e6c45413e8b993e50deb2/transformed/navigation-common-ktx-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/907b04b48f4b93647e8374c4f0285856/transformed/core-ktx-1.3.2/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/edf13d1d369ec6bfd5935d204c8e7673/transformed/transition-1.4.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/15258e48ff5c33f4eeaeceb4449ad053/transformed/navigation-fragment-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/cf20fd632fe42d4671a6baf530553e78/transformed/play-services-ads-identifier-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3cc7f8e04a58dcfeb250b4b7be67ed5d/transformed/firebase-messaging-20.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c6e7b11f6439f4682b0398779bc5d36b/transformed/firebase-iid-20.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/38043d5c5d74f3dec14e2efb5aba1627/transformed/play-services-cloud-messaging-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0e2e41c79d1a2e316fca3157720f16ce/transformed/play-services-stats-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6a35eb2dc17d6fced71898c45d791271/transformed/firebase-datatransport-17.0.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e0d54e7da2b3ed888bdeb91979edb04d/transformed/firebase-installations-16.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0fa057de802823f66bec878f7f0860f6/transformed/firebase-common-19.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4acad1ea353f1ecc18a9bf5cad79ccc5/transformed/firebase-installations-interop-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/276f51d8175ad34843894d268198d7d4/transformed/firebase-iid-interop-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4733c8b38f0608b3e89a306d651e891a/transformed/play-services-fido-18.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2e4a98621f35e8480a9d02c354300813/transformed/play-services-base-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d9845b0d6f08a83f2452d869cc8eed37/transformed/play-services-tasks-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e623264bc1342b3c7f31331900a99ce0/transformed/firebase-measurement-connector-18.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6e9c8cfc96123ff5d1b39da70338d3e9/transformed/play-services-basement-17.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/cfd0a9991ac3c01445e71f0c9b3eba33/transformed/fragment-1.3.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2a9902251d53c99075777af9526df232/transformed/appcompat-resources-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9be78be6b0065bbfb3105cad94e650b8/transformed/browser-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5fbca0f08b8cbc7878ee6c510488f24b/transformed/media-1.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2e719edbb7e80156414efa5eb136071e/transformed/swiperefreshlayout-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b58b3d9f183ce4cd3e53dbae00bf6ce3/transformed/autofill-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/6fd5723aee2ba7e80f4994c5b849bc34/transformed/palette-ktx-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2de9a1667ef1a05c5994656b5803cf96/transformed/palette-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9e10e7e492ba3d404c9862cc250e017f/transformed/legacy-support-core-utils-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2afb91b68ef6cdf1acf1a36e923a81f9/transformed/drawerlayout-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/4a6bca33619975143569ca1ff8c8f929/transformed/viewpager-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/99da73a775c1e60c447661fed4489635/transformed/slidingpanelayout-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0543789f3528b386f27a68cc17e37b2b/transformed/customview-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/edc0fd9c8e79a6d8dac1dc35816bbb45/transformed/work-runtime-2.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/d85e9c74a87ec90781ff57ab6cfe337d/transformed/vectordrawable-animated-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e8b7308f355cb57695ab0c96b7ba690a/transformed/vectordrawable-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c2de62ea70f364e2b3d343b75de5b468/transformed/loader-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f1d8728800449b637ece4005eda06dd9/transformed/asynclayoutinflater-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/320801b521f0453c5fdeba3101d9f171/transformed/navigation-runtime-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/297ec4b74523ef68edfab79e849f2115/transformed/navigation-common-2.3.3/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5c4043fec5bd558006718aee0e4e903a/transformed/activity-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/baad857560f2d4ff00b8865d7ed727d6/transformed/core-1.5.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/24960fddf98576217b2bead3e11f5dcf/transformed/installreferrer-1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/0bbd9961f62d2d50ab567131ef17110a/transformed/core-1.8.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7eff327a33e7987b551f1cf663275bdd/transformed/falcon-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/9a315ffd33d6c9690e8340f55d1ebfe4/transformed/leakcanary-android-core-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ee3d2cbcd1e0e6a785cfbb459aa0db59/transformed/concept-fetch-94.0.20210930211433/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e14626c37a7f24c433c3195e0fa320e4/transformed/ui-util-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/20a55625c06c6b37c5c8d249a712a72c/transformed/lifecycle-livedata-core-ktx-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a93f7da337ed6ee445829abe04db837f/transformed/savedstate-ktx-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/48e056d000755372688f06e73963ee68/transformed/leakcanary-object-watcher-android-androidx-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/eb9dd8cf6304e2a86ecfc62452698816/transformed/leakcanary-object-watcher-android-support-fragments-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/776fab5fbf36040166d8a7b9cae5793a/transformed/leakcanary-object-watcher-android-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2ad8da4aac8c5c06718078f5ae42ef68/transformed/plumber-android-2.4/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5b8b2935491cd32e7ef8f576ad11d97e/transformed/core-ktx-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/dc918cd378f4b0296779ed6753ca63d0/transformed/core-1.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/75e0f7015fabe5db37dd3c6514e3f4fb/transformed/monitor-1.4.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/2ea47e1c6d6558ba78e3075decdd8e57/transformed/cursoradapter-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/356085e61d7c62f307433a2fd22100ac/transformed/lifecycle-viewmodel-savedstate-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/423546c57d677b30e65436d939c1b94a/transformed/savedstate-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/53e7545eef63429183cf5dcf4e852cd4/transformed/lifecycle-service-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b4c3aedb5fce9b1c10ea9983d3b076fe/transformed/lifecycle-runtime-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/451cb01dd8864e5aacdc2822d66e171b/transformed/lifecycle-viewmodel-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/28121fc8a4f55cd5c57a013f9ef91777/transformed/cardview-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3e87ba80ea84d86c3f2422265c87fa52/transformed/localbroadcastmanager-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/a857a917ea921863d507ebf4ef2c7cd7/transformed/profileinstaller-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/df024029ffa7345980aaa2be52cdd9eb/transformed/versionedparcelable-1.1.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/660598e145725505a4069b3186f83fb4/transformed/startup-runtime-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5481c8b693c894066cdc4344331d59e7/transformed/tracing-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/60093caf4c9af68a5f11ee8ff08afada/transformed/lifecycle-livedata-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/3bc690d5184972738662d02e2e236004/transformed/lifecycle-livedata-core-2.3.1/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/770a7de97ffdccd0122e08df65419d61/transformed/room-runtime-2.3.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fe9e2e4af469423ee74cf5f627f235f2/transformed/core-runtime-2.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/c7e8ca8afda093085ca0399eb6bcdc64/transformed/interpolator-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/8b4401888f2b120c267061ce7afadd99/transformed/transport-backend-cct-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/fc200096c4a4e1e717515066b2ed1de2/transformed/transport-runtime-2.2.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/66d8e689d4da23c6f7ff7c0ec057dacb/transformed/firebase-components-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5d34bc4bc1c38336014dec3258cb9209/transformed/firebase-encoders-json-16.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/f7e664fe8b72b9ab5e485369273cec33/transformed/documentfile-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/ad3b5cd9f2583fed60fe5cfa4e25a109/transformed/print-1.0.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/b67b80f3768825d25e8c8cfa53304d87/transformed/sqlite-framework-2.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/7eaef8dbfe59dec6c56a216baefca3ed/transformed/sqlite-2.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/e06f9402bed5dca232cfed6ba191ffe1/transformed/jna-5.8.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/5ec6a40beaa4bdc572d78c7aecbaf7c3/transformed/annotation-experimental-1.1.0/jars/classes.jar, file:/home/runner/.gradle/caches/transforms-3/031dcc11309b63cd09fb5378a04aff8a/transformed/transport-api-2.2.0/jars/classes.jar, file:/home/runner/work/fenix/fenix/app/build/intermediates/compile_and_runtime_not_namespaced_r_class_jar/debug/R.jar, file:/home/runner/.gradle/caches/transforms-3/1ebd8edeb059821eaa6de7c916635ea9/transformed/android.jar])
2021-10-01T02:51:33.9283132Z     at com.sun.jna.Native.extractFromResourcePath(Native.java:1095)
2021-10-01T02:51:33.9299418Z     at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:275)
2021-10-01T02:51:33.9302103Z     ... 38 more
2021-10-01T02:51:33.9303832Z     I [glean/GleanCrashReporterService] No crashes to record, or file not found.
2021-10-01T02:51:33.9306095Z     I [mozac/CrashReporter] Caught Exception report submitted to 1 services
2021-10-01T02:51:33.9307139Z   FAILURE
```